![](_page_0_Picture_0.jpeg)

![](_page_0_Picture_1.jpeg)

**Leon Gehrke 1,†, Vasco Dos Reis Gonçalves 1,† [,](https://orcid.org/0000-0001-9982-3346) Dominik Andrae <sup>1</sup> , Tamas Rasko <sup>1</sup> , Patrick Ho <sup>1</sup> [,](https://orcid.org/0000-0003-4751-6904) Hermann Einsele [1](https://orcid.org/0000-0002-7680-0819) , Michael Hudecek 1,[2](https://orcid.org/0000-0002-2280-2202) and Sabrina R. Friedel 1,[\\*](https://orcid.org/0000-0003-4276-2268)**

- <sup>1</sup> Medizinische Klinik und Poliklinik II und Lehrstuhl für Zelluläre Immuntherapie, Universitätsklinikum Würzburg, 97080 Würzburg, Germany
- <sup>2</sup> Fraunhofer-Institut für Zelltherapie und Immunologie, Außenstelle Zelluläre Immuntherapie, 97070 Würzburg, Germany
- **\*** Correspondence: prommersbe\_s@ukw.de
- † These authors contributed equally to this work.

**Abstract:** The successful application of CAR-T cells in the treatment of hematologic malignancies has fundamentally changed cancer therapy. With increasing numbers of registered CAR-T cell clinical trials, efforts are being made to streamline and reduce the costs of CAR-T cell manufacturing while improving their safety. To date, all approved CAR-T cell products have relied on viral-based gene delivery and genomic integration methods. While viral vectors offer high transfection efficiencies, concerns regarding potential malignant transformation coupled with costly and time-consuming vector manufacturing are constant drivers in the search for cheaper, easier-to-use, safer, and more efficient alternatives. In this review, we examine different non-viral gene transfer methods as alternatives for CAR-T cell production, their advantages and disadvantages, and examples of their applications. Transposon-based gene transfer methods lead to stable but non-targeted gene integration, are easy to handle, and achieve high gene transfer rates. Programmable endonucleases allow targeted integration, reducing the potential risk of integration-mediated malignant transformation of CAR-T cells. Non-integrating CAR-encoding vectors avoid this risk completely and achieve only transient CAR expression. With these promising alternative techniques for gene transfer, all avenues are open to fully exploiting the potential of next-generation CAR-T cell therapy and applying it in a wide range of applications.

**Keywords:** non-viral; gene transfer; CAR-T cells; transposase; sleeping beauty; piggybac; programmable endonuclease; non-integrating; CRISPR; transient

# **1. Introduction**

The successful clinical implementation of chimeric antigen receptor (CAR) T cells depends on the quantity and quality of the drug product, which are strongly affected by the manufacturing process. The critical attributes of CAR-T cell products are safety, purity, consistency, potency, and persistence [\[1\]](#page-16-0). There are many ongoing efforts to replace the manual with automated and centralized point-of-care manufacturing processes, as well as a shift from ex vivo production toward the in vivo generation of CAR-T cells. The development and optimization of innovative technologies, reagents, and devices for delivering genes of interest (GOI) aim to transition the production of CAR-T cells even further in the near future.

Despite major research efforts and advances in recent years, all approved CAR-T cell drug products still use the same complex manufacturing workflows generated specifically for each individual patient. Current state-of-the-art CAR-T cell production involves the isolation of peripheral blood mononuclear cells or T cells from leukapheresis products and their subsequent activation. Afterward, the cells are subjected to gene transfer to express the CAR construct and are expanded for several days to achieve the infusion dose [\[2\]](#page-16-1). Finally, the T cells are harvested, the cell numbers and CAR expression levels are determined,

![](_page_0_Picture_12.jpeg)

**Citation:** Gehrke, L.; Gonçalves, V.D.R.; Andrae, D.; Rasko, T.; Ho, P.; Einsele, H.; Hudecek, M.; Friedel, S.R. Current Non-Viral-Based Strategies to Manufacture CAR-T Cells. *Int. J. Mol. Sci.* **2024**, *25*, 13685. [https://doi.org/](https://doi.org/10.3390/ijms252413685) [10.3390/ijms252413685](https://doi.org/10.3390/ijms252413685)

Academic Editor: Niels Schaft

Received: 19 November 2024 Revised: 12 December 2024 Accepted: 14 December 2024 Published: 21 December 2024

![](_page_0_Picture_16.jpeg)

**Copyright:** © 2024 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license [\(https://](https://creativecommons.org/licenses/by/4.0/) [creativecommons.org/licenses/by/](https://creativecommons.org/licenses/by/4.0/) 4.0/).

and the products are filled according to the required dosage. CAR-T cell production is still a lengthy process that is largely carried out manually or semi-automatically in highly specialized GMP facilities, which results in high costs and limits the production throughput. Commercially approved CAR-T cell products currently have price tags of roughly \$420,000 to 550,000 per batch [\[3\]](#page-16-2). Such high costs are prohibitive to the widespread application of CAR-T cell therapy.

One major cost driver in the clinical production of CAR-T cells is the viral vector or nucleic acid material for gene transfer, which must comply with GMP requirements and fulfill strict quality control specifications. The production of GMP-grade viral vectors is expensive, laborious, and time-consuming and requires S2 laboratories with specially trained personnel. Lot sizes are limited for GMP production, resulting in the need to adjust for lot-to-lot variations during CAR-T cell manufacturing.

All six EMA- and FDA-approved CAR-T cell products rely on viral gene delivery vectors, namely, lentiviral vectors (LVV) or gamma retroviral vectors (γRVV), to achieve gene transfer. Viral vectors are able to introduce payloads of up to 8–10 kb of genetic material into cells. As non-targeted gene transfer systems, lentiviral vectors tend to integrate transgene payloads into actively transcribed genes and transcription start sites, whereas γRVVs frequently integrate into promoter regions, exons, and regulatory elements [\[4,](#page-16-3)[5\]](#page-16-4). This is associated with an increased risk of activation of proto-oncogenes or disruption of tumor-suppressor genes, and therefore, the induction of malignant transformation [\[6](#page-16-5)[,7\]](#page-16-6).

Despite decades of experience with virally produced CAR-T cells, the paradigm is shifting toward the implementation of alternative gene transfer strategies. Besides safety aspects, factors like high production costs, long production times, and regulatory and logistical burdens drive this shift. Transposase systems can achieve high gene transfer rates, while programmable endonucleases can be targeted to specific integration sites, reducing the potential risk of malignant transformation. Non-integrating vectors present opportunities to completely circumvent integrational mutagenesis, and therefore, minimize risks pertaining to the genetic safety of the drug product. Nonetheless, each of these alternative gene transfer systems has limitations that will require further optimization. While the delivery of protein complexes or naked DNA/RNA vectors requires the development of compatible and potent delivery systems able to overcome specific challenges, delivery platforms, like electroporation, cell squeezing, or liponanoparticles, are not the focus of this article and are systematically reviewed elsewhere [\[8](#page-16-7)[,9\]](#page-16-8).

In general, gene transfer should be efficient and lead to a consistent output. It should not affect the vitality, quality, and fitness of the cells, and of course, be safe to use. In addition, factors such as the availability of GMP quality, price, logistics, and batch-to-batch consistency of starting materials have to be considered.

In this review, we discuss current strategies for virus-free gene transfer used in the manufacturing process of CAR-T cells (Figure [1\)](#page-2-0), their advantages and disadvantages, and future perspectives of the field (summarized in Table [1\)](#page-3-0).

<span id="page-2-0"></span>![](_page_2_Figure_2.jpeg)

**Figure 1.** Overview of different strategies for gene transfer. Viruses, transposases, and programmable endonucleases mediate stable integration of the GOI into the genome, and therefore, stable CAR expression. Non-integrating vectors do not induce gene integration and thus induce transient CAR expression as long as the vector is present in the cell. The respective mechanisms and methods of delivery are depicted in a generalized but not necessarily inclusive manner. Transposase protein (blue ellipses); transposon ITRs (red DNA); CAR/GOI (green RNA/DNA/protein); genomic DNA (violet). **Figure 1.** Overview of different strategies for gene transfer. Viruses, transposases, and programmable endonucleases mediate stable integration of the GOI into the genome, and therefore, stable CAR expression. Non-integrating vectors do not induce gene integration and thus induce transient CAR expression as long as the vector is present in the cell. The respective mechanisms and methods of delivery are depicted in a generalized but not necessarily inclusive manner. Transposase protein (blue ellipses); transposon ITRs (red DNA); CAR/GOI (green RNA/DNA/protein); genomic DNA (violet).

**Table 1.** Comparison of different gene transfer methods. **Table 1.** Comparison of different gene transfer methods.

• restricted by tropism

|                                               | Virus<br>Virus<br>•<br>highly efficient                                                                                                                                                                      | Transposase<br>Transposase                                                                                                                                               | Programmable<br>Programmable<br>Endonuclease<br>Endonuclease                                                                                                          | Non-Integrating<br>Non-Integrating<br>Vectors<br>Vectors                                                                                   |
|-----------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
| Delivery<br>Delivery Efficiency<br>Efficiency | •<br>highly efficient (can<br>(can also be used<br>also be used for the<br>for the delivery<br>delivery of other<br>of other<br>described systems)<br>•<br>described<br>restricted by<br>tropism<br>systems) | •<br>intermediate<br>•<br>intermediate<br>(depends on the<br>(depends on the<br>transfection<br>transfection<br>method and<br>method and<br>vector size)<br>vector size) | •<br>intermediate<br>•<br>intermediate<br>(depends on the<br>(depends on the<br>transfection method<br>transfection<br>and vector size)<br>method and<br>vector size) | •<br>highly efficient<br>•<br>highly efficient<br>•<br>mRNA is more<br>•<br>mRNA is more<br>efficient than<br>efficient than<br>DNA<br>DNA |

|                                | Virus                                                                                                                                                                                                       | Transposase                                                                                                                                                                              | Programmable<br>Endonuclease                                                                                                                                                                                                                         | Non-Integrating<br>Vectors                                                                                                                                                                                 |
|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Cargo Size                     | •<br>depends on the<br>virus (up to 7 kb<br>with lentiviruses,<br>36 kb with<br>adenoviruses [10])                                                                                                          | •<br>depends on the<br>transposase (up to<br>100 kb [11])                                                                                                                                | •<br>single nucleotides<br>to multiple kb                                                                                                                                                                                                            | •<br>no packaging<br>limitation<br>•<br>delivery efficiency<br>decreases with<br>increasing<br>construct size                                                                                              |
| Expression<br>Persistence      | •<br>stable expression<br>•<br>undirected genomic<br>integration<br>•<br>prone to integrate<br>close to<br>transcriptional<br>active sites                                                                  | •<br>stable expression<br>•<br>undirected<br>genomic<br>integration<br>•<br>integration<br>pattern depends<br>on the transposase                                                         | •<br>stable expression<br>•<br>directed integration<br>based on NHEJ                                                                                                                                                                                 | •<br>mainly transient<br>expression 1                                                                                                                                                                      |
| Copy Number<br>Control         | •<br>steerable by<br>titrating virus<br>amounts                                                                                                                                                             | •<br>steerable by<br>transposase<br>activity and<br>transposon<br>quantity [12]                                                                                                          | •<br>tightly controllable<br>by targeting<br>mechanism                                                                                                                                                                                               | •<br>n. a.                                                                                                                                                                                                 |
| Cellular and<br>Genomic Safety | •<br>low risk of<br>intracellular<br>detection by STING<br>and TLR<br>•<br>low risk of viral<br>particles being<br>detected by the<br>immune system<br>•<br>potential risk of<br>insertional<br>mutagenesis | •<br>intracellular<br>detection depends<br>on the encoded<br>cDNA<br>•<br>low<br>immunogenicity<br>•<br>potential risk of<br>insertional<br>mutagenesis<br>depends on the<br>transposase | •<br>intracellular<br>detection depends<br>on the encoded<br>cDNA<br>•<br>pre-existing<br>immunity toward<br>Cas9 in a small<br>percentage of the<br>human<br>population [13]<br>•<br>risk of chromosomal<br>rearrangement and<br>off-target effects | •<br>intracellular<br>detection depends<br>on encoded cDNA<br>•<br>low<br>immunogenicity<br>•<br>no risk of<br>genotoxicity when<br>using mRNA<br>•<br>almost no risk of<br>genotoxicity when<br>using DNA |
| Cost                           | •<br>high                                                                                                                                                                                                   | •<br>intermediate                                                                                                                                                                        | •<br>intermediate                                                                                                                                                                                                                                    | •<br>low                                                                                                                                                                                                   |
| Clinical Translation           | •<br>several clinical<br>products approved                                                                                                                                                                  | •<br>well represented<br>in clinical trials                                                                                                                                              | •<br>currently used in<br>one clinical trial [14]                                                                                                                                                                                                    | •<br>well represented<br>in clinical trials                                                                                                                                                                |

#### <span id="page-3-0"></span>**Table 1.** *Cont.*

<sup>1</sup> Very few spontaneous integrations are possible from DNA vectors [\[15\]](#page-16-14). Episomal proliferation is possible in the presence of specific sequences [\[16\]](#page-16-15).

#### **2. Stable Gene Transfer by Transposases**

Transposition is a process by which mobile genetic segments, called transposons or transposable elements, change their positions within a genome. Transposons occur naturally in a wide range of organisms, including bacteria, plants, and animals, and constitute more than 45% of the human genome [\[17\]](#page-16-16). Transposons can be grouped into either class I, called retrotransposons, or class II transposons, also known as DNA transposons [\[18,](#page-16-17)[19\]](#page-16-18).

Retrotransposons use a mechanism in which the transposon DNA segment is first transcribed into an mRNA intermediate and then translated back to DNA by reverse transcriptase, which is often coded as part of the transposable element itself. This newly created DNA segment can then be inserted at another location in the genome [\[18](#page-16-17)[,19\]](#page-16-18).

DNA transposons circumvent the need for an mRNA intermediate or reverse transcriptase using a cut-and-paste mechanism. Here, the transposase enzyme excises transposon DNA segments directly from the donor DNA and reintegrates them at another location within the genome [\[18](#page-16-17)[,19\]](#page-16-18). The two most commonly used DNA transposons for the generation of CAR-T cells are Sleeping Beauty (SB) and PiggyBac (PB), both of which achieve stable, non-targeted gene transfer, as it is achieved with viral vectors.

The transposase and the transposon should be encoded on two separate vectors to reduce the risk of transposase integration into the genome [\[20\]](#page-16-19). This risk can be further mitigated if transposase is encoded as mRNA or delivered as a protein. The amount and ratio of transposase and transposon vectors can influence gene transfer rates and transgene insertion copy numbers [\[12,](#page-16-11)[21\]](#page-16-20). Gene transfer rates and cell viability can be enhanced by replacing conventional plasmids with DNA vectors of reduced size and lacking bacterial backbones, such as minicircles [\[21,](#page-16-20)[22\]](#page-17-0). In clinical applications, in particular, it is recommended that the transposase is encoded on mRNA, leading to a transient expression of that enzyme and, therefore, enhancing the safety of the CAR-T cell product [\[21](#page-16-20)[,23\]](#page-17-1). This reduces the risks of (1) triggering an immune response against CAR-T cells, (2) integrating transposase DNA into the genome, and (3) persistent transposase expression, which allows subsequent re-transpositions to leave footprints across the genome, thus increasing the risk of mutagenesis [\[24–](#page-17-2)[26\]](#page-17-3). The transposase- and transposon-encoding vectors are introduced into the target cells using various delivery techniques, such as electroporation, lipofection, or nanoparticles [\[21](#page-16-20)[,22](#page-17-0)[,27\]](#page-17-4).

## *2.1. Sleeping Beauty*

The class II Tc1/mariner-type transposase Sleeping Beauty (SB) utilizes a cut-and-paste mechanism for gene integration of up to 8 kb in cargo size (Figure [2\)](#page-5-0). The transposon DNA sequence includes the GOI and is flanked on both sides by terminal inverted repeats (TIR) that are 200–250 base pairs in length. Each TIR is bound specifically by two SB transposases forming a homodimer. These homodimers form a paired-end tetrameric complex and excise the transposon from the donor site. The transposases then induce a double-strand break at a random TA position in the genome, at which the excised transposon is integrated [\[28\]](#page-17-5). The excision site is repaired by non-homologous end-joining (NHEJ), leaving a CAG footprint [\[29\]](#page-17-6). When analyzing the insertion sites within the genome, it was found that SB has a potentially safer, close-to-random integration profile compared to viral vectors or PiggyBac (PB) transposase [\[4,](#page-16-3)[21,](#page-16-20)[26,](#page-17-3)[30\]](#page-17-7).

The SB transposase, which occurs naturally in teleost fish where it is transpositionally inactive due to the accumulation of mutations, was first described in 1997 by Zoltan Ivics and colleagues. They used synthetic sequence reconstruction to revive the activity of this enzyme [\[31\]](#page-17-8). Step-by-step, they predicted the original putative transposase sequence and thereby designed, inter alia, the 340 amino acid protein SB10, which proved to be tranpositionally active in vertebrate cells [\[31\]](#page-17-8). Since then, there have been a number of optimizations to further increase the transposition rates when using SB. The transposase itself was optimized through amino acid substitutions [\[32\]](#page-17-9) and high-throughput screening for hyperactive variants [\[33\]](#page-17-10), resulting in the development of a stable, highly soluble SB variant that can be delivered in the protein form [\[30\]](#page-17-7). The transposon vectors were optimized by adjusting the nucleotide residues within the TIRs sequences [\[31](#page-17-8)[,34–](#page-17-11)[36\]](#page-17-12).

There are many examples of the use of SB in the preclinical production of CAR-T cells. As early as 2008, it was shown that CD19-CAR-T cells could be produced by electroporating freshly isolated, non-activated T cells with the SB transposase variants SB10 or SB11, although gene transfer rates were low, and CAR-expressing cells, therefore, had to be enriched [\[37](#page-17-13)[,38\]](#page-17-14). Today, the SB variant SB100x is mainly used for the efficient production of CAR-T cells, CAR natural killer (NK) cells, or CAR cytokine-induced killer (CIK) cells, achieving integration rates of around 50% [\[21,](#page-16-20)[39\]](#page-17-15), 40% [\[40\]](#page-17-16), or 60% [\[41\]](#page-17-17), respectively.

<span id="page-5-0"></span>![](_page_5_Figure_1.jpeg)

**Figure 2.** Transposon-based cut-and-paste gene transfer. Transposon and Transposase are encoded separately. The transposase can be delivered as DNA, mRNA, or protein. A transposon carrying the GOI requires delivery as circular DNA. The SB protein binds to the ITR region of the transposon vector and forms a synaptic complex, in which both ends of the transposon are held together and excised from the DNA vector. For SB, the transposon is integrated at a random TA target site in the host cell genome, resulting in stable expression of the GOI. SB protein (blue); transposon ITRs (red); GOI (green) **Figure 2.** Transposon-based cut-and-paste gene transfer. Transposon and Transposase are encoded separately. The transposase can be delivered as DNA, mRNA, or protein. A transposon carrying the GOI requires delivery as circular DNA. The SB protein binds to the ITR region of the transposon vector and forms a synaptic complex, in which both ends of the transposon are held together and excised from the DNA vector. For SB, the transposon is integrated at a random TA target site in the host cell genome, resulting in stable expression of the GOI. SB protein (blue); transposon ITRs (red); GOI (green).

*2.2. PiggyBac*  PiggyBac (PB) is also a class II transposon that originates from insects and was first described in cabbage looper [52]. It has a size of 594 amino acids and contains DNA-binding domains, a catalytic domain, and dimerization domains [53]. In the transposition process, the PB transposase enzyme binds to the TIR sequences, flanking the transposon, excises the transposon, and integrates it at the TTAA chromosomal sites. PB integration was found to be more frequent near CpG islands, DNAse I hypersensitive sites, and transcriptional starting sites compared to SB integration [54]. Compared to SB, PB has a lower theoretical chance to integrate into genomically safe regions and shows an integration profile resembling that of the MLV retrovirus [4]. PB does not leave a footprint at the excision site and produces no target site duplication [52]. Further, SB was the first transposase used for clinical CAR-T cell manufacturing [\[42](#page-17-18)[,43\]](#page-17-19). In 2016, Kabriaei and colleagues published their phase I studies on the treatment of 26 NHL and ALL patients with second-generation, allogeneic, and autologous CD19-CAR-T cells produced by Sleeping Beauty-based gene transfer. In these clinical trials, T cells were electroporated with two plasmids encoding the CAR-transposon and SB11 transposase and subsequently expanded with antigen-presenting feeder cells. Analysis of the cell products revealed no changes in the TCR repertoire, low VCN of 1 (allogeneic) and 1.3 (autologous), and an integration profile that gave no indication of a common insertional hotspot. The mean CAR expression in CD3+ cells was 83% (allogeneic) and 88.5% (autologous). CAR-T cells proved to be safe and could be detected in an average of 51 days (allogeneic) and 201 days (autologous) in peripheral blood [\[43](#page-17-19)[,44\]](#page-17-20).

As early as 2010, Manuri and colleagues showed that they could use PB transposase for the production of CD19-specific CAR-T cells. Although the gene transfer rate was still extremely low, the number of CAR-expressing cells could be increased to 50% in 21 days with the help of antigen-presenting feeder cells [55]. The same was true for Nakazawa's group, which published the PB-based production of HER2-CAR-T cells one year later. After electroporation of PBMCs with PB transposon SB transposition has additionally been used to produce donor-derived CD19-CARexpressing CIK cells to treat B-ALL after allogeneic hematopoietic stem cell transplantation. CIK cells are artificially generated CD3+ and CD56+ natural killer T cells. For this purpose, PBMCs were electroporated with plasmids encoding a third-generation CD19-CAR and SB11 transposase and differentiated using IFN-y, IL-2, and CD3 mAbs. CAR-CIK production lasted a total of 20–32 days. The mean CAR expression rate was 43%, the mean transgene copy number per cell was 3.5, and a highly polyclonal repertoire of cells was

detected at early time points. Most of the cells produced had a CD8 effector memory phenotype, and 46% were positive for CD56. Robust expansion of CAR-expressing cells was achieved in all patients, and cells could be detected for an average of 94 days [\[45](#page-18-0)[–47\]](#page-18-1).

The CARAMBA clinical trial, in which fresh, autologous SLAMF7-CAR-T cells are used to treat multiple myeloma, was launched in 2021. Here, CD4<sup>+</sup> and CD8<sup>+</sup> CAR-T cells are activated separately by CD3/CD28 stimulation and then electroporated with CAR-encoding minicircle DNA and SB100x-encoding mRNA. The production of the cell product takes 14 days, resulting in a vein-to-vein time of 16 days [\[2\]](#page-16-1).

Further clinical trials based on the SB gene transfer are planned or already ongoing, e.g., one using CD19-CAR-T cells for the treatment of relapsed or refractory B-cell lymphomas (EudraCT 2022-001040-23) and another using ROR1-CAR-T cells for the treatment of hematologic and solid tumors (EudraCT 2022-003728-41).

The application of SB has so far shown great promise, and the full potential of SB gene transfer is yet to be realized in the field of CAR-T cell research. Until now, one of the main shortcomings of the application has been that transposon and transposase are usually introduced into the cells by electroporation—A process that is always accompanied by a drop in cell viability. New methods of gene transport could offer a solution to this problem. However, high gene transfer rates can be achieved with this technique, and SB has been proven to have a more favorable integration profile and lower potential risk of insertional mutagenesis compared to other gene transfer methods [\[4,](#page-16-3)[43](#page-17-19)[,48](#page-18-2)[,49\]](#page-18-3). It offers the possibility of one-step modification of CAR-transfer together with other gene engineering techniques, e.g., CRISPR gene editing [\[50](#page-18-4)[,51\]](#page-18-5). Furthermore, SB has already been successfully tested in automated manufacturing devices [\[39\]](#page-17-15), and the production of starting materials is less costly and less complicated compared to that of viral gene transfer.

#### *2.2. PiggyBac*

PiggyBac (PB) is also a class II transposon that originates from insects and was first described in cabbage looper [\[52\]](#page-18-6). It has a size of 594 amino acids and contains DNA-binding domains, a catalytic domain, and dimerization domains [\[53\]](#page-18-7). In the transposition process, the PB transposase enzyme binds to the TIR sequences, flanking the transposon, excises the transposon, and integrates it at the TTAA chromosomal sites. PB integration was found to be more frequent near CpG islands, DNAse I hypersensitive sites, and transcriptional starting sites compared to SB integration [\[54\]](#page-18-8). Compared to SB, PB has a lower theoretical chance to integrate into genomically safe regions and shows an integration profile resembling that of the MLV retrovirus [\[4\]](#page-16-3). PB does not leave a footprint at the excision site and produces no target site duplication [\[52\]](#page-18-6).

As early as 2010, Manuri and colleagues showed that they could use PB transposase for the production of CD19-specific CAR-T cells. Although the gene transfer rate was still extremely low, the number of CAR-expressing cells could be increased to 50% in 21 days with the help of antigen-presenting feeder cells [\[55\]](#page-18-9).

The same was true for Nakazawa's group, which published the PB-based production of HER2-CAR-T cells one year later. After electroporation of PBMCs with PB transposon and transposase vectors, the observed gene transfer rates were low, but CAR-expressing T cells could be enriched to around 40% by magnetic selection and expansion [\[56\]](#page-18-10).

Efforts to enable the clinical use of PB-generated CAR-T cells have gained traction [\[57](#page-18-11)[,58\]](#page-18-12), and the transposition process has been steadily optimized by developing hyperactive PB mutants, reducing the lengths of TIR sequences, and replacing conventional plasmids with smaller DNA vectors [\[59](#page-18-13)[–61\]](#page-18-14).

However, this development slowed in 2021 with the publication by Micklethwaite and colleagues, who reported the appearance of malignant CAR-T cells in the first-in-human study involving CD19-CAR-T cells produced by PB transposition. In this study, known as the CAR-TELL trial, CD19-CAR-T cells were used to treat B-cell malignancies after allogeneic hematopoietic stem cell transfer (HSCT). For this purpose, the CAR-transposon plasmid and PB mRNA were electroporated into T cells from the allogeneic HSCT donor, achieving high gene transfer rates of around 72%. The release tests for unwanted autonomous T cell growth remained unremarkable. Nevertheless, two of the 10 treated patients developed CAR-positive T cell lymphomas. The first patient had received three doses of CAR-T cells from his HLA-matched sibling donor and developed T cell lymphoma of the effector T helper type 1 subtype. The malignant cells had a relatively high transgene copy number (24), but this was not increased compared to non-malignant cells of the same product. Whole genome sequencing of the monoclonal cells revealed insertions within genes and within 2.5 kb of the transcription starting site, but no insertion into commonly known oncogenes. Transcriptome analysis showed transgene promoter-driven upregulation of sequences at the insertion sites and indicated transcriptional read-through processes of up to 1000 kb. These read-through events included non-coding intronic and out-of-frame sequences, as well as in-frame exon expression of four genes. In addition, widespread copy number gains and losses were observed in tumor-suppressor genes and oncogenes. The second patient exhibited monoclonal CAR-expressing CD8<sup>+</sup> T cells. The transgene copy number was only four, but read-through events were again detectable. The cells of both patients had insertions in the intron of the BACH2 gene, which led to its reduced expression. Since there were some abnormalities but no obvious known trigger described, the cause of the transformation could not be clearly determined [\[62,](#page-18-15)[63\]](#page-18-16).

Until these triggers have been clearly identified and a strategy to prevent malignant transformation has been developed, research on PB-generated CAR-T cells will certainly continue with the handbrake on.

## *2.3. Other Transposases*

Two other transposases that have been used in the context of CAR-T cell manufacturing are transposable element of Oryzias latipes, number 2 (Tol2) [\[64\]](#page-18-17) originating from medakafish [\[65\]](#page-18-18) and TcBuster [\[66\]](#page-18-19), which was originally isolated from the red flour beetle [\[67\]](#page-19-0).

Different isoforms of Tol2 exist with different levels of activity [\[68\]](#page-19-1). The transposition efficiency of Tol2 is increased by codon optimization and shortening of the transposable element [\[69\]](#page-19-2). At 10–11 kb, the cargo capacity of Tol2 is high enough for comprehensive T cell modification [\[69](#page-19-2)[,70\]](#page-19-3). Tol2 preferentially integrates at AT-rich palindrome-like DNA regions and in transcription units, mostly in introns [\[71\]](#page-19-4). It generates DNA duplications of eight base pairs and leaves footprints after excision [\[72\]](#page-19-5).

TcBuster can transfer large cargo with sizes of 10–12 kb. A hyperactive mutant was generated using high-throughput screening of 3 million variants, significantly increasing gene transfer rates in human NK and T cells [\[66\]](#page-18-19). Integration site profiling revealed a safer integration profile compared to lentiviral vectors but a greater tendency to integrate into transcripts, exons, transcription start sites, CpG islands, and DNAse I cleavage sites compared to SB [\[66](#page-18-19)[,73\]](#page-19-6). TcBuster induces eight base pair target site duplications, inducing footprints at excision sites [\[67\]](#page-19-0).

#### **3. Targeted Integration Using Programmable Endonucleases**

Targeted integration of GOI can be achieved by co-delivery of programmable endonucleases, which induce double-strand breaks (DSBs) at selected sites in the genome, along with transgenes encoded within DNA repair template vectors (Figure [3\)](#page-8-0). The key component is a programmable nuclease, which relies on RNA molecules or DNA-binding protein domains to recognize and cut the targeted genome site. The characteristics of the supplied DNA donor template help to instruct the mechanism of repair, thus ensuring the precision of the integration.

![](_page_8_Figure_1.jpeg)

**3. Targeted Integration Using Programmable Endonucleases** 

<span id="page-8-0"></span>precision of the integration.

Targeted integration of GOI can be achieved by co-delivery of programmable endonucleases, which induce double-strand breaks (DSBs) at selected sites in the genome, along with transgenes encoded within DNA repair template vectors (Figure 3). The key component is a programmable nuclease, which relies on RNA molecules or DNA-binding protein domains to recognize and cut the targeted genome site. The characteristics of the supplied DNA donor template help to instruct the mechanism of repair, thus ensuring the

**Figure 3.** Targeted transgene integration using double-strand break induction via programmable nucleases: Genomic DNA containing the targeted sequence is cleaved by protein-DNA interactions or RNA-guided endonucleases. The resulting double-strand break (DSB) is repaired either by the error-prone non-homologous end-joining (NHEJ) pathway or by homology-directed repair (HDR). This results in correct repair or insertions and deletions (INDELs). Supplying a single- or doublestranded DNA donor template carrying homologous sequences can facilitate precise integration of the GOI at the target locus. Concurrent delivery and cleavage of a non-homologue DNA donor template can facilitate non-directional targeted integration. Targeted genomic DNA sequence (violet); GOI (green); INDEL (red with halo). **Figure 3.** Targeted transgene integration using double-strand break induction via programmable nucleases: Genomic DNA containing the targeted sequence is cleaved by protein-DNA interactions or RNA-guided endonucleases. The resulting double-strand break (DSB) is repaired either by the errorprone non-homologous end-joining (NHEJ) pathway or by homology-directed repair (HDR). This results in correct repair or insertions and deletions (INDELs). Supplying a single- or double-stranded DNA donor template carrying homologous sequences can facilitate precise integration of the GOI at the target locus. Concurrent delivery and cleavage of a non-homologue DNA donor template can facilitate non-directional targeted integration. Targeted genomic DNA sequence (violet); GOI (green); INDEL (red with halo).

#### *3.1. Programmable Endonucleases: RNA-Guided Nucleases 3.1. Programmable Endonucleases: RNA-Guided Nucleases*

The discovery of the Clustered regularly interspaced short palindromic repeats associated with the Cas endonucleases (CRISPR-Cas) system has revolutionized the field of gene therapy due to its unparalleled versatility, cost-effectiveness, and ease of use. Engineered CRISPR-Cas systems, most prominently implementing variants of CRISPR-associated (Cas) nuclease 9, offer a highly flexible two-component system combining an endonuclease and an easy-to-design short guide RNA (sgRNA) for selective targeting. The The discovery of the Clustered regularly interspaced short palindromic repeats associated with the Cas endonucleases (CRISPR-Cas) system has revolutionized the field of gene therapy due to its unparalleled versatility, cost-effectiveness, and ease of use. Engineered CRISPR-Cas systems, most prominently implementing variants of CRISPR-associated (Cas) nuclease 9, offer a highly flexible two-component system combining an endonuclease and an easy-to-design short guide RNA (sgRNA) for selective targeting. The sgRNA mediates DNA-binding via Watson-Crick base-pairing to instruct the specific targeting of virtually any defined sequence within the genome. The only prerequisite is the presence of the protospacer-adjacent motif (PAM), a three-nucleotide sequence (NGG) immediately adjacent to the freely selectable, 20 nucleotide (nt) long recognition sequence that directs binding to the specified target locus. Incremental improvements to the Cas9 endonuclease have allowed for the generation of higher specificity or higher fidelity variants like the aptly named Cas9 Sniper [\[74\]](#page-19-7) and Cas9HiFi [\[75\]](#page-19-8), respectively, effectively expanding the CRISPR/Cas9 toolbox. Additionally, other engineered variants offer solutions to broaden the range of genomic sequences potentially targetable by Cas9 by relaxing the sequence restrictions (Cas9 SpG, NGN) or almost entirely removing PAM requirements (Cas9 SpRY, NRN/NYN) [\[76\]](#page-19-9). Notably, Cas9 was used for the original study, which showed a CAR

sequence being inserted within the T cell receptor alpha constant (TRAC) gene to generate TCR-deficient knock-in CAR-T cells [\[77\]](#page-19-10).

Other RNA-guided nucleases in and outside the Cas family have expanded the available toolset. The most prominent RNA-guided agent, aside from Cas9, is Cas12/CPF1 and its variants [\[78,](#page-19-11)[79\]](#page-19-12). The combination of the two systems enables multiplexed orthogonal editing without guide switching. Multiplexed transgene delivery has already been demonstrated, generating multiplex-edited T cells expressing CAR within the TRAC locus and silencing human leukocyte antigen HLA I and HLA II expression [\[80\]](#page-19-13).

## *3.2. Programmable Endonucleases: Protein-DNA Interaction-Guided Nucleases*

Zinc-finger Nucleases (ZFNs), Transcription Activator-Like Effector Nucleases (TAL-ENs), and Meganucleases are the most prominent programmable nucleases that use proteinbased recognition of DNA motifs to mediate specificity. The protein domain integrity necessary for correct folding and subsequent DNA binding of these nucleases limits the number of potentially targetable sequences in the genome, but the long history of usage provides an advanced understanding of targeting mechanisms and off-target effects.

Meganucleases are naturally occurring monomeric endonucleases that are able to specifically recognize long DNA sequences (>18 nt) and mediate cleavage. Accordingly, retargeting requires work-intensive combinatorial re-engineering of the complete protein and can be restricted to a limited set of targetable sequences.

ZFNs and TALENs are recombinant nucleases that use DNA-binding protein domains in combination with a FokI Nuclease Domain. Both ZFNs and TALENs are designed as monomeric proteins that cut one strand of DNA and act in dimers to induce DSBs. To achieve the juxtaposed orientation necessary to generate a DSB at the same position, each pair of TALENs or ZFNs requires two distinct target sequences, and DSB induction requires successful dimerization subsequent to the independent recognition of both sequences. This enables well-designed ZFN and TALENs to cut DNA with high specificity and reduce the risk of off-target effects. ZFN derive their specificity from combinations of zinc-finger (ZF) domains that recognize three bases each. State-of-the-art ZFN specificity is conveyed by the specific recognition of at least 18 nucleotides. The retargeting of ZFNs is a timeconsuming engineering process that is further complicated by non-targetable nucleotide triplet combinations, reciprocal influences between single ZF domains, and cytotoxicity [\[81\]](#page-19-14).

For TALENs, DNA binding is mediated by combining different repeat-variable diresidues (RVDs) polymorphic positions in highly conserved amino acid repeats that specifically recognize and bind one nucleotide [\[82\]](#page-19-15). The combination of RVDs matching the four DNA bases creates DNA-binding domains that recognize 18–20 nt, lifting target sequence restrictions and allowing for comparatively easy retargeting [\[83\]](#page-19-16). While all nucleases are predominantly used for combinatorial editing in combination with other strategies, including viruses for CAR delivery, protein-DNA targeting has proven to be a flexible and effective technology, even for sophisticated CAR-T cell engineering approaches, including targeted CAR integration [\[84,](#page-19-17)[85\]](#page-19-18).

#### *3.3. Transgene Integration via Cellular DSB-Repair Mechanisms*

Innate eukaryotic surveillance and repair mechanisms aim to maintain genomic stability and sequence integrity by preventing long-lasting DNA damage [\[86\]](#page-19-19). Targeted transgene integration can be achieved by hijacking endogenous cellular DNA repair mechanisms that are activated upon the occurrence of a double-strand break.

Repair pathways can either be homology-dependent or-independent, with the choice and outcome of the repair mechanism being a highly regulated, multifactorial process instructed via DNA end configuration, sequence identity, spatial alignment, and input from several central pathways regulating the cell cycle or homeostasis. Both the most prominent somatic DNA damage repair pathways, non-homologous end-joining (NHEJ) and homology-directed repair (HDR), can be exploited to integrate transgenes into open break sites [\[87\]](#page-19-20).

HDR mechanisms are potentially error-free multi-step repair processes that allow for template-based repair, fixing the DSB via templated synthesis of a complementary DNA strand and subsequent recombination [\[88\]](#page-19-21). In somatic cells, synthesis-dependent strand annealing (SDSA), which results in conservative repair without crossover, is the predominant HDR repair pathway and is active during the S and G2 phases of the cell cycle. Substitute pathways include single-strand annealing (SSA) or alternative end-joining by microhomology-mediated template switching (MMEJ) [\[89\]](#page-19-22). By designing DNA donor vectors to flank the transgene with homology arms matching the sequences upstream and downstream of the DSB, the endogenous repair machinery can facilitate precise transgene integration using endogenous HDR pathways. The single-strand or double-strand DNA donor template carrying the transgene can vary in size and end-modification, with efficient integration of several kb long constructs (e.g., CARs) being the state-of-the-art [\[77\]](#page-19-10). Building on homology repair, short single-stranded templates can be used to introduce single nucleotide changes or corrections of short stretches of up to 50 base pairs. This strategy has been exploited to facilitate the integration of larger transgenes, such as CARs, via primed microhomologues-assisted integration (PAINT) [\[90,](#page-19-23)[91\]](#page-19-24).

In contrast, NHEJ is a rapid, high-capacity repair pathway that requires no or minimal template sequence for reference and acts independently of the cell cycle state on open DNA ends, representing the default repair mechanism upon spontaneous DSBs [\[92\]](#page-19-25). This repair pathway joins open DNA ends in an error-prone manner, resulting in the resolution of DSBs with insertions or deletions (INDELs) in approximately 70% of repaired double-stranded DNA [\[93\]](#page-20-0). NHEJ is often taken advantage of to induce INDELs with programmable endonucleases to knock out relevant genes by disrupting the endogenously encoded genetic information. It can further be used to facilitate homology-independent targeted integration (HITI) of transgenes via concurrent donor and genomic target DNA cleavage in a sitespecific and efficient, albeit non-directional, INDEL-prone, and less precise manner [\[94\]](#page-20-1).

The limiting factors for efficient targeted integration approaches using HDR are usually attributed to DNA-induced toxicities, which increase with the total DNA amount and length of the transgene, and the relatively low natural HDR frequencies, which are insufficient for cell product manufacturing [\[95\]](#page-20-2). Recent developments aim to increase the efficiency of HDR-mediated transgene integration by (1) synchronizing cell cycle states prior to gene editing [\[96\]](#page-20-3), (2) modifying the HDR template to either facilitate interaction with HDR-related proteins [\[97\]](#page-20-4), or (3) hiding the template DNA from triggering innate immune pathways that sense and respond to foreign DNA, thus reducing the cytotoxicity experienced by target cells during gene delivery. The state-of-the-art strategy to increase HDR-mediated donor integration is to administer small molecules that inhibit key factors in the NHEJ pathway, such as the DNA-dependent phosphokinase catalytic subunit (DNA-PKc), which is rapidly recruited to the DSB ends and facilitates a crucial, deterministic step to initiate NHEJ repair. In this way, the DNA-PKc–treated cells are forced to rely on the remaining HDR-mediated repair pathways or otherwise suffer apoptosis due to genotoxicity [\[98–](#page-20-5)[100\]](#page-20-6).

#### *3.4. Strategic Application and Risk Mitigation*

Targeted integration allows for loci-specific delivery of transgenes, making loci-effects, including 3D chromatin structure, chromatin accessibility, and transgene copy number predictable and selectable. Several loci are considered to be "safe harbor" sites, such as adeno-associated virus site 1 (AAVS1), protein phosphatase 1 regulatory subunit 12 C (PPP1R12C), or chemokine receptor 5 (CCR5), all of which have been shown to be clinically safe loci for targeted transgene integration. The specific disruption of these loci has not revealed any detrimental impact on cell differentiation or function, thus ensuring that integration at these sites has no relevant negative effects on the potency of the edited cells [\[101\]](#page-20-7).

Further, the simultaneous disruption of targeted loci upon (CAR) transgene integration can even be advantageous for certain cell engineering approaches. T cell receptor (TCR)- KO CAR-T cells mitigating graft-versus-host disease have proven useful for non-targeted CAR delivery and approaches using concurrent CAR integration into the TCR-alpha-chain locus [\[102\]](#page-20-8). Concurrent KO of safe-harbor locus CCR5 during CAR integration yields HIV-resistant CAR-T cells [\[103\]](#page-20-9), and KO of immune checkpoint receptors allows for the generation of CAR-T cells that are not impeded by inhibitory signals within suppressive tumor micro environments [\[103\]](#page-20-9). Combinatorial approaches can be used to eliminate HLA expression, relieving the requirement for donor HLA matching. These multi-target editing strategies can be employed in combination with different modes of CAR delivery to generate allogeneic "universal" or "of-the-shelf" CAR-T cells [\[104\]](#page-20-10). Linking CAR expression to TCR signaling via site-specific integration of CARs into the human TRAC locus has been associated with activation-induced downregulation of CAR expression, improved therapy outcome, and reduced exhaustion of T cells after antigen encounter [\[105\]](#page-20-11). The potential of such sophisticated strategies is illustrated by the engineering of receptors that directly incorporate the desired endogenous protein parts via integration at the endogenous locus. Prominently, the CD247-domain-deficient CAR, integrated into the CD247 locus, allows for the generation of CAR-T and CAR-NK cells, that express the CAR under control of the endogenous promoter of CD247 and incorporate the endogenously encoded signaling domain into the final surface protein [\[106\]](#page-20-12). Contrary to TRAC-KI approaches, this strategy enables the generation of CAR-NK cells expressing CAR under an endogenous promoter active in NK cells and simultaneously reduces the cargo size upon donor delivery. Intricate engineering approaches at this level have also been used to create new engineering-based receptor classes like HLA-independent T cell (HIT) receptors [\[106](#page-20-12)[,107\]](#page-20-13).

Although site-specific genome-editing technologies have enabled cell engineering with unprecedented precision, the applications to CAR-T cell manufacturing continue to face significant challenges in feasibility and safety. The low scalability and inefficiency of gene delivery pose problems for cell products that require large quantities of genome-editing material. Additionally, detrimental effects resulting from genetic editing (or editing-related processes) raise genomic safety concerns [\[95\]](#page-20-2). Off-target DNA cleavage can disrupt unrelated genes or even lead to larger translocations [\[108\]](#page-20-14). Several genome-wide technologies have been developed to predict and detect intended and unintended editing outcomes for in silico, in vitro, and in vivo applications. Guide and Circle Seq are the most prominent tools for detecting off-target aberrations [\[109,](#page-20-15)[110\]](#page-20-16). CAST-Seq focuses on large translocations [\[111\]](#page-20-17) recently employed to re-evaluate the safety profiles of base editors and TALENs [\[112\]](#page-20-18). The recently presented MEGA dPCR relies on digital droplet PCR (ddPCR) and aims to integrate most on- and off-target assessments into one assay [\[113\]](#page-20-19). Currently, a diverse assortment of methods needs to be applied to estimate and subsequently mitigate the genomic effects of nucleases. Furthermore, the real-world frequency of these genomic abberations, as well as their relevance (e.g., transformative mutations), remains hard to assess.

Due to the risk of off-target editing and imperfect in silico prediction algorithms, it is necessary to make an individual risk assessment for each new target and each new CRISPRguide that is used for the generation of CAR-T cells. Currently, the need to extensively evaluate the safety, efficacy, and individual off-target effects has resulted in new clinical trials for products with minor changes in the targeting agent [\[114,](#page-20-20)[115\]](#page-20-21) (see FDA guidance for industry on human gene therapy products incorporating human genome editing from January 2024).

Another significant challenge to the widespread application of CAR-T cells generated by targeted integration is the current limitation of DNA donor templates. The need for DNA donors excludes many modern gene editing delivery approaches that rely on delivering all used agents, either as less immunogenic RNA or directly as proteins. This largely restricts the truly virus-free application of ex vivo approaches for delivering agents via nucleofection. In clinical applications, delivery via non-integrating AAV Vectors is usually needed to facilitate high editing efficiency and allow for simultaneous delivery of the DNA donor template, making it a not truly virus-free strategy and raising the respective costs of the therapies due to expensive vector production. Novel strategies to overcome this hurdle are being developed [\[94\]](#page-20-1).

The restrictions imposed by safety requirements and transgene delivery options make CAR-T cells generated by targeted insertion best suited for the exogenous production of allogeneic T cells. In this setting, multiplexed, targeted genome editing facilitates the development of universal or off-the-shelf CAR-T cells via the simultaneous disruption of genes involved in HLA-mismatch–induced rejection. Allogenic CAR-T cell therapies hold great translational promise as off-the-shelf products. However, while gene-edited CAR-T cells are already in clinical use, there have been few trials of CAR-T cells using gene editing for CAR delivery [\[14\]](#page-16-13).

#### **4. Gene Transfer by Non-Integrating Vectors**

A promising alternative for gene delivery in CAR-T cell immunotherapy is the use of non-integrating strategies. DNA- or mRNA-based vectors can deliver GOIs without requiring the co-delivery of any genomic editing machinery and have the potential to reduce both the cost and length of manufacturing protocols while bypassing the risks of insertional oncogenesis, genomic translocations, and rearrangements [\[116,](#page-21-0)[117\]](#page-21-1). To date, the major challenge for these vectors stems from the transient nature of gene transfer to engineered cells. However, with different half-lives, transformation with both DNA and mRNA results in transient transgene expression, effectively abolishing the capacity for long-term CAR-T cell expansion and persistence. Advancements in the design and structure of both DNA and RNA vectors that enable improved persistence or even vector self-replication have the potential to circumvent these aspects and renew interest in such technologies.

#### *4.1. DNA-Based Vectors*

Bacterial-derived plasmids have historically been the most frequently used DNAbased vectors in cancer gene therapy. Their easy design, coupled with low cost of manufacturing, high stability, and long shelf life, are the advantages that make these strategies attractive. The advent of minicircles and nanoplasmids, which greatly reduced the vector backbone size (from the traditional 2 kb observed in plasmids to below 500 bps) and the amount of bacterial-derived sequences, further addressed challenges like bacterial backbone-induced GOI silencing, toxicity to target cells, and cellular and metabolic disturbance. These improvements are especially important in the context of immunotherapy. T cells are characterized by their extreme sensitivity to immunogenic exogenous DNA, which induces rapid functional loss or even apoptosis [\[118\]](#page-21-2). Electroporation with DNA minicircles has been shown to reduce gene transfer-associated toxicity while simultaneously resulting in higher transfection efficiencies when compared to conventional plasmids.

However, due to the need for an in vivo site-specific recombination step coupled with chromatography-based purification, minicircle production has a low scalability potential characterized by low manufacturing yields. Nanoplasmids, which implement a ~500 bp RNA-out selection marker [\[119,](#page-21-3)[120\]](#page-21-4), offer the same transfection benefits as minicircles while implementing different manufacturing setups [\[121\]](#page-21-5).

Continuous, systematic refinement of DNA vector composition has allowed for the development of non-integrating lentiviral-derived vectors (NILV) [\[122\]](#page-21-6) capable of nuclear extra-chromosomal replication. Such constructs make use of the scaffold/matrix attachment region (S/MAR) elements [\[123\]](#page-21-7), effectively allowing for the generation of persistently modified host cells [\[16](#page-16-15)[,124\]](#page-21-8). CD19-CAR-T cells engineered with these NILV-S/MAR vectors display similar levels of CAR expression and on-target in vivo cytotoxic capacity and expansion when compared to LV-engineered T cells [\[125\]](#page-21-9). Importantly, the insertion of NILV-S/MAR vectors into the genome of host cells was below the limit of detection.

More recently, Bozza and colleagues further refined this technology by developing the minimally sized nS/MARt DNA nanovector, characterized by its low toxicity when used for the transfection of T cells [\[126\]](#page-21-10). nS/MARt DNA nanovectors contain no viral components and have been used for the successful generation of in vitro and in vivo functional CAR-T cells [\[118\]](#page-21-2).

#### *4.2. mRNA-Based Vectors*

mRNA-based therapies have ushered in a new era in modern medicine and play a substantial role in the current immunotherapy clinical trial landscape [\[127](#page-21-11)[,128\]](#page-21-12). Since mRNA's main functions are performed in the cytoplasm, effectively bypassing the need to be shuttled to the nucleus, RNA-based therapies offer advantages over DNA-based ones. High transfection efficiencies and low toxicities observed with mRNA-based vectors have refreshed interest in ultra-fast manufacturing protocols. Most strikingly, efficient delivery of mRNA can be achieved by milder delivery methods like cell squeezing technology or liponanoparticles (LNP), allowing for increased viability of the cell product. Such technologies are typically not efficient for DNA delivery, specifically due to the lack of nuclear delivery capacity. Additionally, mRNA production is more convenient, rapid and economical when compared to DNA synthesis. Currently, in vitro transcription (IVT) reactions are preferred over direct chemical synthesis, allowing for higher fidelity in the production of longer mRNA sequences coupled with higher yields of the mRNA product [\[129,](#page-21-13)[130\]](#page-21-14). Since IVT reactions result in a mixture of both single- and double-stranded mRNA, purification steps are necessary to reduce the immunogenicity of the mRNA product, as double-stranded RNA is a potent activator of cytosolic innate immune sensors [\[131\]](#page-21-15).

Recent developments in the mRNA field have allowed for systematic and essential enhancements regarding (a) increased stability in the form of 5′ and 3′ untranslated region optimization [\[132,](#page-21-16)[133\]](#page-21-17), (b) increased translatability by the incorporation of synthetic caps [\[134](#page-21-18)[–136\]](#page-21-19), poly A tail length [\[137](#page-21-20)[–139\]](#page-21-21), as well as codon and secondary structure optimization [\[130,](#page-21-14)[140,](#page-21-22)[141\]](#page-21-23), and (c) decreased immunogenicity by the incorporation of alternative nucleotides like pseudouridine [\[142](#page-21-24)[,143\]](#page-22-0).

Nonetheless, mRNA molecules are short-lived and result in transient expression of the GOI. In clinical trials, CAR transcript detection by post-infusion quantitative PCR demonstrated a rapid decline in GOI expression, with undetectable levels reached after 1–3 days [\[144,](#page-22-1)[145\]](#page-22-2). Although the inherent kinetics of mRNA can be advantageous for exploring targets with a high risk of toxicity, it can also make the generation of several doses per patient a logistically and commercially prohibitive process [\[144\]](#page-22-1). A clinical trial with mRNA-delivered CD123-CAR-T cells reported failure to manufacture 40% of its planned doses and was eventually terminated [\[146\]](#page-22-3). With some patients needing up to six infusions of 1E8 CAR-T cells, as reported in some clinical trials [\[145,](#page-22-2)[147\]](#page-22-4), failure to generate a drug product can cap the feasibility and commercialization of therapy. Additionally, mRNA is inherently less stable than DNA-based vectors, which means that each additional drug dose correlates with additional costs related to GMP production, drug product storage, and eventual distribution.

Additionally, in another mRNA CAR-T cell trial, frequent administration of the drug product led to an adverse anaphylactic reaction in one of the three patients treated [\[148](#page-22-5)[,149\]](#page-22-6). Although no other mRNA-based CAR-T cell clinical trial has reported anaphylaxis, care should be taken regarding the burden of repeated infusions to the patient.

New emerging technologies, such as mRNA delivery systems for sustained release [\[150,](#page-22-7)[151\]](#page-22-8), self-amplifying RNAs [\[152\]](#page-22-9), and circular RNA [\[153\]](#page-22-10), all aim to overcome transient GOI expression issues but have yet to reach clinical trials [\[154](#page-22-11)[,155\]](#page-22-12).

#### **5. Future Perspectives**

Four decades after the first antibody/TCR chimeric molecules were first described [\[156,](#page-22-13)[157\]](#page-22-14), six CAR-T cell products have reached the market, with many more being tested for their safety and efficacy in a wide spectrum of disease types. The therapeutic results observed against hematologic malignancies confirm the potential of CAR-T cells in the field of immunotherapy. To date, all approved products are manufactured by viral gene

transfer, although non-viral techniques represent a promising alternative for CAR-T cell manufacturing in the future.

It is important to keep in mind that one of the most decisive factors that determine the suitability of a gene transfer method is its safety regarding the potential to induce genotoxicity. Overall, the risk of secondary T cell malignancies occurring after CAR-T cell therapy is rather low [\[158\]](#page-22-15), and the mechanisms leading to it, or whether they are induced by CAR gene transfer, are still unknown [\[62,](#page-18-15)[159\]](#page-22-16). However, there is at least a potential risk of genotoxicity when using non-targeted integrating vectors. Gene destruction or dysregulation can be caused by transgene integration into active host genes or their regulatory elements. Furthermore, promoters within the transgene itself can alter the expression of genes near the integration site. If this occurs, especially within proto-oncogenes or tumor-suppressor genes, malignant transformation of CAR-T cells could be observed [\[63,](#page-18-16)[160\]](#page-22-17).

Although not a targeted integration method, SB transposition has been shown to have a safer, and therefore more preferential, integration profile compared to viral vectors or PB transposition in terms of targeted transcriptional units [\[4](#page-16-3)[,21\]](#page-16-20). Additionally, transposonbased gene transfer is a cost-effective and easy-to-handle method for producing CAR-T cells. The production of starting materials (DNA and mRNA vectors) that meet clinical quality standards is easily scalable and subject to little variation compared with the production of viral vectors. Gene transfer rates and cell viability have been further optimized, especially by replacing common plasmid vectors with smaller DNA alternatives, such as minicircles [\[21\]](#page-16-20), allowing for efficiencies comparable to those of viral vectors. The risks of re-mobilization, integration of the transposase expression cassette, and induction of immunogenicity can be reduced by the use of mRNA encoding the transposase [\[21\]](#page-16-20).

A promising technical improvement to achieve stable therapeutic transgene genomic integration and expression is the combination of a non-integrating adeno-associated virus (AAV) and the Tc1/mariner transposon system. This has resulted in hybrid vectors that exploit the highly efficient targeting properties of the virus through infection and harness stable transposase-mediated gene integration of the transgene in the host cell. These hybrid systems achieved a high rate of stable gene transfer into host cells and, in the case of Sleeping Beauty, possessed a similar safe gene integration profile as previously verified in different mammalian cell lines [\[161–](#page-22-18)[165\]](#page-22-19).

RNA-based retroelements, which are class I transposons, could be a future alternative. In 2024, Zhang et al. described the approach of precise RNA-mediated insertion of transgenes (PRINT), in which mRNA vectors are combined with sequence-specific integration. Using this technique, transgenes can be directly synthesized by reverse transcription into the genome at a multicopy safe-harbor locus, inducing stable and safe integration [\[166\]](#page-23-0).

Precision editing, enabled by diverse nuclease technologies and targeted integration approaches, has allowed researchers to engineer cells with unprecedented levels of customization and sophistication. However, although targeted integration approaches using programmable nucleases circumvent the insertional mutagenesis risks associated with (semi-) random integrating vectors, this class of proteins comes with other aspects of genotoxicity that can be attributed to its nuclease activity. Screening technology is rapidly developing to meet the field's need for genotoxicity and risk assessments. Simultaneously, ongoing advances in nuclease and editing technology are rapidly improving off-target and on-target performances. For example, the development of new techniques, such as baseediting or prime editing, which enable precise nucleotide substitutions, has the potential to mediate edits with even higher precision, and the latter has already been proven capable of safe and targeted introduction of larger transgenes [\[167,](#page-23-1)[168\]](#page-23-2).

The integration efficiency of current technologies has continuously improved, reaching levels comparable to those of other methods. Nonetheless, the requirement for DNA donors and the respective limitations in the choice of delivery strategy currently exclude efficient in vivo as well as truly virus-free, large-scale manufacturing approaches. Future solutions may allow for enrichment and reduced genotoxicity or altogether eliminate the need for viral delivery, thus sidestepping the costs related to GMP virus production [\[94\]](#page-20-1). Until then,

CAR-T cells generated using nuclease-mediated targeted integration are best suited for approaches aiming to generate off-the-shelf or universal CAR-T cells where the product can be subjected to intense engineering and rigorous safety screening prior to administration without time constraints [\[169\]](#page-23-3).

While the temporary expression of genome-manipulating proteins can limit undesirable toxicities, effectively improving the safety profile of these gene editing systems [\[2](#page-16-1)[,170\]](#page-23-4), mRNA can be used for the transient delivery and expression of the CAR molecule itself [\[66\]](#page-18-19). Paired with the lack of genomic integration, high transfection efficiencies, and affordable costs, mRNA vectors offer improved safety profiles that match well with the interest in in vivo generation of CAR-T cells.

Regardless of the non-viral gene transfer system used, directed delivery strategies need to be refined for safe and efficient in vitro or in vivo gene transfer. While gene delivery via viral gene transfer is achieved by natural cellular mechanisms, non-viral gene transfer success is dependent on the development and advancement of concomitant delivery techniques. Until now, most laboratories have used electroporation for the ex vivo transfer of mRNA, DNA, or proteins into cells, a technique that always requires a compromise between a high gene transfer rate and cell viability. Nanoparticles, like LNPs, could be an attractive alternative for the gene delivery of non-viral vectors. These diverse biomaterials offer improved stability of reagents with the option to encapsulate not only proteins but also DNA- or RNA-based vectors, with the possibility of targeting specific cell populations [\[171\]](#page-23-5). LNPs have consistently demonstrated their applicability not only in the context of CRISPR-Cas9, emphasizing their potential for the delivery of nucleic acids in immunotherapies [\[94,](#page-20-1)[172\]](#page-23-6), but also in mRNA vaccines by protecting mRNA from nuclease degradation [\[173\]](#page-23-7). In the future, T cell-targeted, antibody-functionalized nanoparticles [\[172\]](#page-23-6) could be used for the in vivo generation of CAR-T cells, completely bypassing the very expensive and cumbersome ex vivo production in a GMP laboratory.

Overall, the method of gene transfer influences the effectiveness, safety, and accessibility of cell products. It must meet high standards in terms of safety, efficiency, scalability, availability, handling, and cost, among others. It is, therefore, of great importance to further develop and optimize alternative techniques and to regularly weigh their advantages and disadvantages against prevalent virus-based gene transfer methods.

**Author Contributions:** Conceptualization, S.R.F., L.G. and V.D.R.G.; writing—original draft preparation, S.R.F., L.G. and V.D.R.G.; writing—review and editing, D.A., T.R., P.H., H.E. and M.H.; visualization, S.R.F. and L.G.; supervision, M.H. and H.E. All authors have read and agreed to the published version of the manuscript.

**Funding:** This research received no external funding.

**Acknowledgments:** The authors acknowledge funding by the German Ministry for Science and Education (BMBF, Bundesministerium für Bildung und Forschung, grant #13N15986), the German Research Foundation (Deutsche Forschungsgemeinschaft, DFG, TRR338, TRR221) and Stiftung Forschung hilft. The authors are members of the National Center for Tumor Diseases (NCT) WERA. MH and SRF are members of the SaxoCell consortium. SRF received funding from the DKMS John Hansen Research Grant. The authors are supported by the European Union's innovation program under grant agreements No 101016909 (AIDPATH to MH & HE, No 754658 (CARAMBA to MH, HE & SRF), and Innovative Medicines Initiative 2 Joint Undertaking under grant agreements No 853988 (imSAVAR to MH & HE) and No 945393 (T2EVOLVE to MH & HE). This Joint Undertaking receives support from the European Union's Horizon 2020 research and innovation program and EFPIA. Figures were created in [https://BioRender.com.](https://BioRender.com)

**Conflicts of Interest:** MH and SRF are listed as inventors of patent applications and granted patents related to CAR-T technologies that have been filed by the Fred Hutchinson Cancer Research Center, Seattle, WA, and by the University of Würzburg, Würzburg, Germany. MH is the co-founder and equity owner of T-CURX GmbH, Würzburg, Germany. VG and SRF hold secondary employment at T-CURX GmbH. MH received honoraria from Celgene/BMS, Janssen, and Kite/Gilead. HE received honoraria from Pfizer, Amgen, Janssen, Sanofi, and BMS. The remaining authors declare that their research was conducted in the absence of any commercial or financial relationships that could be construed as potential conflicts of interest.

## **References**

- <span id="page-16-0"></span>1. Reddy, O.L.; Stroncek, D.F.; Panch, S.R. Improving CAR T cell therapy by optimizing critical quality attributes. *Semin. Hematol.* **2020**, *57*, 33–38. [\[CrossRef\]](https://doi.org/10.1053/j.seminhematol.2020.07.005)
- <span id="page-16-1"></span>2. Prommersberger, S.; Reiser, M.; Beckmann, J.; Danhof, S.; Amberger, M.; Quade-Lyssy, P.; Einsele, H.; Hudecek, M.; Bonig, H.; Ivics, Z. CARAMBA: A first-in-human clinical trial with SLAMF7 CAR-T cells prepared by virus-free Sleeping Beauty gene transfer to treat multiple myeloma. *Gene Ther.* **2021**, *28*, 560–571. [\[CrossRef\]](https://doi.org/10.1038/s41434-021-00254-w) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/33846552)
- <span id="page-16-2"></span>3. Cliff, E.R.S.; Kelkar, A.H.; Russler-Germain, D.A.; Tessema, F.A.; Raymakers, A.J.N.; Feldman, W.B.; Kesselheim, A.S. High Cost of Chimeric Antigen Receptor T-Cells: Challenges and Solutions. *Am. Soc. Clin. Oncol. Educ. Book* **2023**, *43*, e397912. [\[CrossRef\]](https://doi.org/10.1200/EDBK_397912)
- <span id="page-16-3"></span>4. Gogol-Döring, A.; Ammar, I.; Gupta, S.; Bunse, M.; Miskey, C.; Chen, W.; Uckert, W.; Schulz, T.F.; Izsvák, Z.; Ivics, Z. Genome-wide Profiling Reveals Remarkable Parallels Between Insertion Site Selection Properties of the MLV Retrovirus and the piggyBac Transposon in Primary Human CD4+ T Cells. *Mol. Ther.* **2016**, *24*, 592–606. [\[CrossRef\]](https://doi.org/10.1038/mt.2016.11)
- <span id="page-16-4"></span>5. Shao, L.; Shi, R.; Zhao, Y.; Liu, H.; Lu, A.; Ma, J.; Cai, Y.; Fuksenko, T.; Pelayo, A.; Shah, N.N.; et al. Genome-wide profiling of retroviral DNA integration and its effect on clinical pre-infusion CAR T-cell products. *J. Transl. Med.* **2022**, *20*, 514. [\[CrossRef\]](https://doi.org/10.1186/s12967-022-03729-5)
- <span id="page-16-5"></span>6. Hacein-Bey-Abina, S.; Von Kalle, C.; Schmidt, M.; McCormack, M.P.; Wulffraat, N.; Leboulch, P.; Lim, A.; Osborne, C.S.; Pawliuk, R.; Morillon, E.; et al. LMO2-Associated Clonal T Cell Proliferation in Two Patients after Gene Therapy for SCID-X1. *Science* **2003**, *302*, 415–419. [\[CrossRef\]](https://doi.org/10.1126/science.1088547) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/14564000)
- <span id="page-16-6"></span>7. Fraietta, J.A.; Nobles, C.L.; Sammons, M.A.; Lundh, S.; Carty, S.A.; Reich, T.J.; Cogdill, A.P.; Morrissette, J.J.D.; DeNizio, J.E.; Reddy, S.; et al. Disruption of TET2 promotes the therapeutic efficacy of CD19-targeted T cells. *Nature* **2018**, *558*, 307–312. [\[CrossRef\]](https://doi.org/10.1038/s41586-018-0178-z) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/29849141)
- <span id="page-16-7"></span>8. Taghdiri, M.; Mussolino, C. Viral and Non-Viral Systems to Deliver Gene Therapeutics to Clinical Targets. *Int. J. Mol. Sci.* **2024**, *25*, 7333. [\[CrossRef\]](https://doi.org/10.3390/ijms25137333)
- <span id="page-16-8"></span>9. Alsaiari, S.K.; Eshaghi, B.; Du, B.; Kanelli, M.; Li, G.; Wu, X.; Zhang, L.; Chaddah, M.; Lau, A.; Yang, X.; et al. CRISPR–Cas9 delivery strategies for the modulation of immune and non-immune cells. *Nat. Rev. Mater.* **2024**. [\[CrossRef\]](https://doi.org/10.1038/s41578-024-00725-7)
- <span id="page-16-9"></span>10. Lee, C.S.; Bishop, E.S.; Zhang, R.; Yu, X.; Farina, E.M.; Yan, S.; Zhao, C.; Zeng, Z.; Shu, Y.; Wu, X.; et al. Adenovirus-mediated gene delivery: Potential applications for gene and cell-based therapies in the new era of personalized medicine. *Genes Dis.* **2017**, *4*, 43–63. [\[CrossRef\]](https://doi.org/10.1016/j.gendis.2017.04.001)
- <span id="page-16-10"></span>11. Rostovskaya, M.; Fu, J.; Obst, M.; Baer, I.; Weidlich, S.; Wang, H.; Smith, A.J.H.; Anastassiadis, K.; Stewart, A.F. Transposonmediated BAC transgenesis in human ES cells. *Nucleic Acids Res.* **2012**, *40*, e150. [\[CrossRef\]](https://doi.org/10.1093/nar/gks643) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/22753106)
- <span id="page-16-11"></span>12. Grabundzija, I.; Irgang, M.; Mátés, L.; Belay, E.; Matrai, J.; Gogol-Döring, A.; Kawakami, K.; Chen, W.; Ruiz, P.; Chuah, M.K.L.; et al. Comparative Analysis of Transposable Element Vector Systems in Human Cells. *Mol. Ther.* **2010**, *18*, 1200–1209. [\[CrossRef\]](https://doi.org/10.1038/mt.2010.47)
- <span id="page-16-12"></span>13. Wagner, D.L.; Amini, L.; Wendering, D.J.; Burkhardt, L.-M.; Akyüz, L.; Reinke, P.; Volk, H.-D.; Schmueck-Henneresse, M. High prevalence of Streptococcus pyogenes Cas9-reactive T cells within the adult human population. *Nat. Med.* **2019**, *25*, 242–248. [\[CrossRef\]](https://doi.org/10.1038/s41591-018-0204-6)
- <span id="page-16-13"></span>14. Zhang, J.; Hu, Y.; Yang, J.; Li, W.; Zhang, M.; Wang, Q.; Zhang, L.; Wei, G.; Tian, Y.; Zhao, K.; et al. Non-viral, specifically targeted CAR-T cells achieve high safety and efficacy in B-NHL. *Nature* **2022**, *609*, 369–374. [\[CrossRef\]](https://doi.org/10.1038/s41586-022-05140-y) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/36045296)
- <span id="page-16-14"></span>15. Wang, Z.; Troilo, P.J.; Wang, X.; Griffiths, T.G.; Pacchione, S.J.; Barnum, A.B.; Harper, L.B.; Pauley, C.J.; Niu, Z.; Denisova, L.; et al. Detection of integration of plasmid DNA into host genomic DNA following intramuscular injection and electroporation. *Gene Ther.* **2004**, *11*, 711–721. [\[CrossRef\]](https://doi.org/10.1038/sj.gt.3302213)
- <span id="page-16-15"></span>16. Piechaczek, C.; Fetzer, C.; Baiker, A.; Bode, J.; Lipps, H.J. A vector based on the SV40 origin of replication and chromosomal S/MARs replicates episomally in CHO cells. *Nucleic Acids Res.* **1999**, *27*, 426–428. [\[CrossRef\]](https://doi.org/10.1093/nar/27.2.426) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/9862961)
- <span id="page-16-16"></span>17. Lander, E.S.; Linton, L.M.; Birren, B.; Nusbaum, C.; Zody, M.C.; Baldwin, J.; Devon, K.; Dewar, K.; Doyle, M.; FitzHugh, W.; et al. Initial sequencing and analysis of the human genome. *Nature* **2001**, *409*, 860–921. [\[CrossRef\]](https://doi.org/10.1038/35057062)
- <span id="page-16-17"></span>18. Finnegan, D. Eukariotic Transposable Elements and Genome Evolution. *Trends Genet.* **1989**, *5*, 103–107. [\[CrossRef\]](https://doi.org/10.1016/0168-9525(89)90039-5)
- <span id="page-16-18"></span>19. Wicker, T.; Sabot, F.; Hua-Van, A.; Bennetzen, J.L.; Capy, P.; Chalhoub, B.; Flavell, A.; Leroy, P.; Morgante, M.; Panaud, O.; et al. A unified classification system for eukaryotic transposable elements. *Nat. Rev. Genet.* **2007**, *8*, 973–982. [\[CrossRef\]](https://doi.org/10.1038/nrg2165) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/17984973)
- <span id="page-16-19"></span>20. Saha, S.; Woodard, L.E.; Charron, E.M.; Welch, R.C.; Rooney, C.M.; Wilson, M.H. Evaluating the potential for undesired genomic effects of the piggyBac transposon system in human cells. *Nucleic Acids Res.* **2015**, *43*, 1770–1782. [\[CrossRef\]](https://doi.org/10.1093/nar/gkv017) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/25605795)
- <span id="page-16-20"></span>21. Monjezi, R.; Miskey, C.; Gogishvili, T.; Schleef, M.; Schmeer, M.; Einsele, H.; Ivics, Z.; Hudecek, M. Enhanced CAR T-cell engineering using non-viral Sleeping Beauty transposition from minicircle vectors. *Leukemia* **2017**, *31*, 186–194. [\[CrossRef\]](https://doi.org/10.1038/leu.2016.180)
- <span id="page-17-0"></span>22. Pastor, M.; Johnen, S.; Harmening, N.; Quiviger, M.; Pailloux, J.; Kropp, M.; Walter, P.; Ivics, Z.; Izsvák, Z.; Thumann, G.; et al. The Antibiotic-free pFAR4 Vector Paired with the Sleeping Beauty Transposon System Mediates Efficient Transgene Delivery in Human Cells. *Mol. Ther. Nucleic Acids* **2018**, *11*, 57–67. [\[CrossRef\]](https://doi.org/10.1016/j.omtn.2017.12.017) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/29858090)
- <span id="page-17-1"></span>23. Miskey, C.; Amberger, M.; Reiser, M.; Prommersberger, S.; Beckmann, J.; Machwirth, M.; Einsele, H.; Hudecek, M.; Bonig, H.; Ivics, Z. Genomic Analyses of SLAMF7 CAR-T Cells Manufactured by Sleeping Beauty Transposon Gene Transfer for Immunotherapy of Multiple Myeloma. *bioRxiv* **2019**, 675009. [\[CrossRef\]](https://doi.org/10.1101/675009)
- <span id="page-17-2"></span>24. Wilber, A.; Wangensteen, K.J.; Chen, Y.; Zhuo, L.; Frandsen, J.L.; Bell, J.B.; Chen, Z.J.; Ekker, S.C.; McIvor, R.S.; Wang, X. Messenger RNA as a source of transposase for Sleeping Beauty transposon-mediated correction of hereditary tyrosinemia type I. *Mol. Ther.* **2007**, *15*, 1280–1287. [\[CrossRef\]](https://doi.org/10.1038/sj.mt.6300160)
- 25. Huang, X.; Haley, K.; Wong, M.; Guo, H.; Lu, C.; Wilber, A.; Zhou, X. Unexpectedly high copy number of random integration but low frequency of persistent expression of the sleeping beauty transposase after trans delivery in primary human T cells. *Hum. Gene Ther.* **2010**, *21*, 1577–1590. [\[CrossRef\]](https://doi.org/10.1089/hum.2009.138)
- <span id="page-17-3"></span>26. Liang, Q.; Kong, J.; Stalker, J.; Bradley, A. Chromosomal mobilization and reintegration of Sleeping Beauty and PiggyBac transposons. *Genesis* **2009**, *47*, 404–408. [\[CrossRef\]](https://doi.org/10.1002/dvg.20508) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/19391106)
- <span id="page-17-4"></span>27. Rodgers, T.; Muzzio, N.; Watson, C.; Romero, G. Stabilization of poly (β-amino ester) nanoparticles for the efficient intracellular delivery of piggybac transposon. *Bioengineering* **2021**, *8*, 16. [\[CrossRef\]](https://doi.org/10.3390/bioengineering8020016) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/33498466)
- <span id="page-17-5"></span>28. Ochmann, M.T.; Ivics, Z. Jumping ahead with sleeping beauty: Mechanistic insights into cut-and-paste transposition. *Viruses* **2021**, *13*, 76. [\[CrossRef\]](https://doi.org/10.3390/v13010076) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/33429848)
- <span id="page-17-6"></span>29. Ivics, Z.; Izsvák, Z. Sleeping Beauty Transposition. *Microbiol. Spectr.* **2015**, *3*, 851–872. [\[CrossRef\]](https://doi.org/10.1128/microbiolspec.MDNA3-0042-2014)
- <span id="page-17-7"></span>30. Querques, I.; Mades, A.; Zuliani, C.; Miskey, C.; Alb, M.; Grueso, E.; Machwirth, M.; Rausch, T.; Einsele, H.; Ivics, Z.; et al. A highly soluble Sleeping Beauty transposase improves control of gene insertion. *Nat. Biotechnol.* **2019**, *37*, 1502–1512. [\[CrossRef\]](https://doi.org/10.1038/s41587-019-0291-z)
- <span id="page-17-8"></span>31. Ivics, Z.; Hackett, P.B.; Plasterk, R.H.; Izsvák, Z. Molecular Reconstruction of Sleeping Beauty, a Tc1-like Transposon from Fish, and Its Transposition in Human Cells. *Cell* **1997**, *91*, 501–510. [\[CrossRef\]](https://doi.org/10.1016/S0092-8674(00)80436-5) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/9390559)
- <span id="page-17-9"></span>32. Geurts, A.M.; Yang, Y.; Clark, K.J.; Liu, G.; Cui, Z.; Dupuy, A.J.; Bell, J.B.; Largaespada, D.A.; Hackett, P.B. Gene transfer into genomes of human cells by the sleeping beauty transposon system. *Mol. Ther.* **2003**, *8*, 108–117. [\[CrossRef\]](https://doi.org/10.1016/S1525-0016(03)00099-6)
- <span id="page-17-10"></span>33. Mátés, L.; Chuah, M.K.L.; Belay, E.; Jerchow, B.; Manoj, N.; Acosta-Sanchez, A.; Grzela, D.P.; Schmitt, A.; Becker, K.; Matrai, J.; et al. Molecular evolution of a novel hyperactive Sleeping Beauty transposase enables robust stable gene transfer in vertebrates. *Nat. Genet.* **2009**, *41*, 753–761. [\[CrossRef\]](https://doi.org/10.1038/ng.343) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/19412179)
- <span id="page-17-11"></span>34. Cui, Z.; Geurts, A.M.; Liu, G.; Kaufman, C.D.; Hackett, P.B. Structure-function analysis of the inverted terminal repeats of the Sleeping Beauty transposon. *J. Mol. Biol.* **2002**, *318*, 1221–1235. [\[CrossRef\]](https://doi.org/10.1016/S0022-2836(02)00237-1) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/12083513)
- 35. Izsvák, Z.; Khare, D.; Behlke, J.; Heinemann, U.; Plasterk, R.H.; Ivics, Z. Involvement of a bifunctional, paired-like DNA-binding domain and a transpositional enhancer in Sleeping Beauty transposition. *J. Biol. Chem.* **2002**, *277*, 34581–34588. [\[CrossRef\]](https://doi.org/10.1074/jbc.M204001200)
- <span id="page-17-12"></span>36. Yant, S.R.; Park, J.; Huang, Y.; Mikkelsen, J.G.; Kay, M.A. Mutational Analysis of the N-Terminal DNA-Binding Domain of Sleeping Beauty Transposase: Critical Residues for DNA Binding and Hyperactivity in Mammalian Cells. *Mol. Cell. Biol.* **2004**, *24*, 9239–9247. [\[CrossRef\]](https://doi.org/10.1128/MCB.24.20.9239-9247.2004)
- <span id="page-17-13"></span>37. Huang, X.; Guo, H.; Kang, J.; Choi, S.; Zhou, T.C.; Tammana, S.; Lees, C.J.; Li, Z.Z.; Milone, M.; Levine, B.L.; et al. Sleeping beauty transposon-mediated engineering of human primary T cells for therapy of CD19+ lymphoid malignancies. *Mol. Ther.* **2008**, *16*, 580–589. [\[CrossRef\]](https://doi.org/10.1038/sj.mt.6300404)
- <span id="page-17-14"></span>38. Singh, H.; Manuri, P.R.; Olivares, S.; Dara, N.; Dawson, M.J.; Huls, H.; Hackett, P.B.; Kohn, D.B.; Shpall, E.J.; Champlin, R.E.; et al. Redirecting specificity of T-cell populations for CD19 using the sleeping beauty system. *Cancer Res.* **2008**, *68*, 2961–2971. [\[CrossRef\]](https://doi.org/10.1158/0008-5472.CAN-07-5600)
- <span id="page-17-15"></span>39. Lock, D.; Monjezi, R.; Brandes, C.; Bates, S.; Lennartz, S.; Teppert, K.; Gehrke, L.; Karasakalidou-Seidt, R.; Lukic, T.; Schmeer, M.; et al. Automated, scaled, transposon-based production of CAR T cells. *J. Immunother. Cancer* **2022**, *10*, e005189. [\[CrossRef\]](https://doi.org/10.1136/jitc-2022-005189)
- <span id="page-17-16"></span>40. Bexte, T.; Botezatu, L.; Miskey, C.; Gierschek, F.; Moter, A.; Wendel, P.; Reindl, L.M.; Campe, J.; Villena-Ossa, J.F.; Gebel, V.; et al. Engineering of potent CAR NK cells using non-viral Sleeping Beauty transposition from minimalistic DNA vectors. *Mol. Ther.* **2024**, *32*, 2357–2372. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2024.05.022)
- <span id="page-17-17"></span>41. Rotiroti, M.C.; Buracchi, C.; Arcangeli, S.; Galimberti, S.; Valsecchi, M.G.; Perriello, V.M.; Rasko, T.; Alberti, G.; Magnani, C.F.; Cappuzzello, C.; et al. Targeting CD33 in Chemoresistant AML Patient-Derived Xenografts by CAR-CIK Cells Modified with an Improved SB Transposon System. *Mol. Ther.* **2020**, *28*, 1974–1986. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2020.05.021)
- <span id="page-17-18"></span>42. Kebriaei, P.; Huls, H.; Jena, B.; Munsell, M.; Jackson, R.; Lee, D.A.; Hackett, P.B.; Rondon, G.; Shpall, E.; Champlin, R.E.; et al. Infusing CD19-directed T cells to augment disease control in patients undergoing autologous hematopoietic stem-cell transplantation for advanced B-lymphoid malignancies. *Hum. Gene Ther.* **2012**, *23*, 444–450. [\[CrossRef\]](https://doi.org/10.1089/hum.2011.167)
- <span id="page-17-19"></span>43. Kebriaei, P.; Singh, H.; Huls, M.H.; Figliola, M.J.; Bassett, R.; Olivares, S.; Jena, B.; Dawson, M.J.; Kumaresan, P.R.; Su, S.; et al. Phase i trials using sleeping beauty to generate CD19-specific CAR T cells. *J. Clin. Invest.* **2016**, *126*, 3363–3376. [\[CrossRef\]](https://doi.org/10.1172/JCI86721) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/27482888)
- <span id="page-17-20"></span>44. Singh, H.; Srour, S.A.; Milton, D.R.; McCarty, J.; Dai, C.; Gaballa, M.R.; Ammari, M.; Olivares, S.; Huls, H.; De Groot, E.; et al. Sleeping beauty generated CD19 CAR T-Cell therapy for advanced B-Cell hematological malignancies. *Front. Immunol.* **2022**, *13*, 1032397. [\[CrossRef\]](https://doi.org/10.3389/fimmu.2022.1032397) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/36439104)
- <span id="page-18-0"></span>45. Magnani, C.F.; Mezzanotte, C.; Cappuzzello, C.; Bardini, M.; Tettamanti, S.; Fazio, G.; Cooper, L.J.N.; Dastoli, G.; Cazzaniga, G.; Biondi, A.; et al. Preclinical Efficacy and Safety of CD19CAR Cytokine-Induced Killer Cells Transfected with Sleeping Beauty Transposon for the Treatment of Acute Lymphoblastic Leukemia. *Hum. Gene Ther.* **2018**, *29*, 602–613. [\[CrossRef\]](https://doi.org/10.1089/hum.2017.207)
- 46. Magnani, C.F.; Tettamanti, S.; Alberti, G.; Pisani, I.; Biondi, A. Transposon-Based CAR T Cells in Acute Leukemias. *Cells* **2020**, *9*, 1337. [\[CrossRef\]](https://doi.org/10.3390/cells9061337)
- <span id="page-18-1"></span>47. Magnani, C.F.; Gaipa, G.; Lussana, F.; Belotti, D.; Gritti, G.; Napolitano, S.; Matera, G.; Cabiati, B.; Buracchi, C.; Borleri, G.; et al. Sleeping Beauty–engineered CAR T cells achieve antileukemic activity without severe toxicities. *J. Clin. Invest.* **2020**, *130*, 6021–6033. [\[CrossRef\]](https://doi.org/10.1172/JCI138473) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/32780725)
- <span id="page-18-2"></span>48. Holstein, M.; Mesa-Nuñez, C.; Miskey, C.; Almarza, E.; Poletti, V.; Schmeer, M.; Grueso, E.; Ordóñez Flores, J.C.; Kobelt, D.; Walther, W.; et al. Efficient Non-viral Gene Delivery into Human Hematopoietic Stem Cells by Minicircle Sleeping Beauty Transposon Vectors. *Mol. Ther.* **2018**, *26*, 1137–1153. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2018.01.012) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/29503198)
- <span id="page-18-3"></span>49. Moldt, B.; Staunstrup, N.H.; Jakobsen, M.; Yáñez-Muñoz, R.J.; Mikkelsen, J.G. Genomic insertion of lentiviral DNA circles directed by the yeast Flp recombinase. *BMC Biotechnol.* **2008**, *8*, 60. [\[CrossRef\]](https://doi.org/10.1186/1472-6750-8-60) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/18691430)
- <span id="page-18-4"></span>50. Tipanee, J.; Samara-Kuko, E.; Gevaert, T.; Chuah, M.K.; VandenDriessche, T. Universal allogeneic CAR T cells engineered with Sleeping Beauty transposons and CRISPR-CAS9 for cancer immunotherapy. *Mol. Ther.* **2022**, *30*, 3155–3175. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2022.06.006) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/35711141)
- <span id="page-18-5"></span>51. Calviño, C.; Ceballos, C.; Alfonso, A.; Jauregui, P.; Calleja-Cervantes, M.E.; San Martin-Uriz, P.; Rodriguez-Marquez, P.; Martin-Mallo, A.; Iglesias, E.; Abizanda, G.; et al. Optimization of universal allogeneic CAR-T cells combining CRISPR and transposon-based technologies for treatment of acute myeloid leukemia. *Front. Immunol.* **2023**, *14*, 1270843. [\[CrossRef\]](https://doi.org/10.3389/fimmu.2023.1270843) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37795087)
- <span id="page-18-6"></span>52. Fraser, M.J.; Smith, G.E.; Summers, M.D. Acquisition of Host Cell DNA Sequences by Baculoviruses: Relationship Between Host DNA Insertions and FP Mutants of Autographa californica and Galleria mellonella Nuclear Polyhedrosis Viruses. *J. Virol.* **1983**, *47*, 287–300. [\[CrossRef\]](https://doi.org/10.1128/jvi.47.2.287-300.1983)
- <span id="page-18-7"></span>53. Sarkar, A.; Sim, C.; Hong, Y.S.; Hogan, J.R.; Fraser, M.J.; Robertson, H.M.; Collins, F.H. Molecular evolutionary analysis of the widespread piggyBac transposon family and related "domesticated" sequences. *Mol. Genet. Genomics* **2003**, *270*, 173–180. [\[CrossRef\]](https://doi.org/10.1007/s00438-003-0909-0) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/12955498)
- <span id="page-18-8"></span>54. Huang, X.; Guo, H.; Tammana, S.; Jung, Y.C.; Mellgren, E.; Bassi, P.; Cao, Q.; Tu, Z.J.; Kim, Y.C.; Ekker, S.C.; et al. Gene transfer efficiency and genome-wide integration profiling of sleeping beauty, Tol2, and PiggyBac transposons in human primary t cells. *Mol. Ther.* **2010**, *18*, 1803–1813. [\[CrossRef\]](https://doi.org/10.1038/mt.2010.141) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/20606646)
- <span id="page-18-9"></span>55. Manuri, P.V.R.; Wilson, M.H.; Maiti, S.N.; Mi, T.; Singh, H.; Olivares, S.; Dawson, M.J.; Huls, H.; Lee, D.A.; Rao, P.H.; et al. PiggyBac transposon/transposase system to generate CD19-specific T cells for the treatment of B-lineage malignancies. *Hum. Gene Ther.* **2010**, *21*, 427–437. [\[CrossRef\]](https://doi.org/10.1089/hum.2009.114) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/19905893)
- <span id="page-18-10"></span>56. Nakazawa, Y.; Huye, L.E.; Salsman, V.S.; Leen, A.M.; Ahmed, N.; Rollins, L.; Dotti, G.; Gottschalk, S.M.; Wilson, M.H.; Rooney, C.M. PiggyBac-mediated cancer immunotherapy using EBV-specific cytotoxic T-cells expressing HER2-specific chimeric antigen receptor. *Mol. Ther.* **2011**, *19*, 2133–2143. [\[CrossRef\]](https://doi.org/10.1038/mt.2011.131)
- <span id="page-18-11"></span>57. Saito, S.; Tanaka, M.; Hasegawa, A.; Inada, Y.; Morokawa, H.; Usami, Y.; Kojima, S.; Yanagisawa, R.; Miyazawa, C.; Ito, K.; et al. A Phase I/II Clinical Trial of Piggybac -Modified GMR CAR-T Cell Therapy for CD116 Positive Relapsed/Refractory Myeloid Malignancies. *Blood* **2021**, *138*, 4813. [\[CrossRef\]](https://doi.org/10.1182/blood-2021-149089)
- <span id="page-18-12"></span>58. Zhang, Y.; Zhang, Z.; Ding, Y.; Fang, Y.; Wang, P.; Chu, W.; Jin, Z.; Yang, X.; Wang, J.; Lou, J.; et al. Phase I clinical trial of EGFR-specific CAR-T cells generated by the piggyBac transposon system in advanced relapsed/refractory non-small cell lung cancer patients. *J. Cancer Res. Clin. Oncol.* **2021**, *147*, 3725–3734. [\[CrossRef\]](https://doi.org/10.1007/s00432-021-03613-7) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/34032893)
- <span id="page-18-13"></span>59. Hua, W.K.; Hsu, J.C.; Chen, Y.C.; Chang, P.S.; Wen, K.L.K.; Wang, P.N.; Yang, W.C.; Shen, C.N.; Yu, Y.S.; Chen, Y.C.; et al. Quantum pBac: An effective, high-capacity piggyBac-based gene integration vector system for unlocking gene therapy potential. *FASEB J.* **2023**, *37*, 1–19. [\[CrossRef\]](https://doi.org/10.1096/fj.202201654R)
- 60. Yusa, K.; Zhou, L.; Li, M.A.; Bradley, A.; Craig, N.L. A hyperactive piggyBac transposase for mammalian applications. *Proc. Natl. Acad. Sci. USA* **2011**, *108*, 1531–1536. [\[CrossRef\]](https://doi.org/10.1073/pnas.1008322108)
- <span id="page-18-14"></span>61. Cadiñanos, J.; Bradley, A. Generation of an inducible and optimized piggyBac transposon system. *Nucleic Acids Res.* **2007**, *35*, e87. [\[CrossRef\]](https://doi.org/10.1093/nar/gkm446) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/17576687)
- <span id="page-18-15"></span>62. Micklethwaite, K.P.; Gowrishankar, K.; Gloss, B.S.; Li, Z.; Street, J.A.; Moezzi, L.; Mach, M.A.; Sutrave, G.; Clancy, L.E.; Bishop, D.C.; et al. Investigation of product-derived lymphoma following infusion of piggyBac-modified CD19 chimeric antigen receptor T cells. *Blood* **2021**, *138*, 1391–1405. [\[CrossRef\]](https://doi.org/10.1182/blood.2021010858) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/33974080)
- <span id="page-18-16"></span>63. Bishop, D.C.; Clancy, L.E.; Simms, R.; Burgess, J.; Mathew, G.; Moezzi, L.; Street, J.A.; Sutrave, G.; Atkins, E.; McGuire, H.M.; et al. Development of CAR T-cell lymphoma in 2 of 10 patients effectively treated with piggyBac -modified CD19 CAR T cells. *Blood* **2021**, *138*, 1504–1509. [\[CrossRef\]](https://doi.org/10.1182/blood.2021010813) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/34010392)
- <span id="page-18-17"></span>64. Tsukahara, T.; Iwase, N.; Kawakami, K.; Iwasaki, M.; Yamamoto, C.; Ohmine, K.; Uchibori, R.; Teruya, T.; Ido, H.; Saga, Y.; et al. The Tol2 transposon system mediates the genetic engineering of T-cells with CD19-specific chimeric antigen receptors for B-cell malignancies. *Gene Ther.* **2015**, *22*, 209–215. [\[CrossRef\]](https://doi.org/10.1038/gt.2014.104) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/25427612)
- <span id="page-18-18"></span>65. Koga, A.; Suzuki, M.; Inagaki, H.; Bessho, Y.; Hori, H. Transposable element in fish. *Nature* **1996**, *383*, 30. [\[CrossRef\]](https://doi.org/10.1038/383030a0) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/8779712)
- <span id="page-18-19"></span>66. Skeate, J.G.; Pomeroy, E.J.; Slipek, N.J.; Jones, B.J.; Wick, B.J.; Chang, J.-W.; Lahr, W.S.; Stelljes, E.M.; Patrinostro, X.; Barnes, B.; et al. Evolution of the clinical-stage hyperactive TcBuster transposase as a platform for robust non-viral production of adoptive cellular therapies. *Mol. Ther.* **2024**, *32*, 1817–1834. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2024.04.024) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/38627969)
- <span id="page-19-0"></span>67. Arensburger, P.; Hice, R.H.; Zhou, L.; Smith, R.C.; Tom, A.C.; Wright, J.A.; Knapp, J.; O'Brochta, D.A.; Craig, N.L.; Atkinson, P.W. Phylogenetic and functional characterization of the hAT transposon superfamily. *Genetics* **2011**, *188*, 45–57. [\[CrossRef\]](https://doi.org/10.1534/genetics.111.126813)
- <span id="page-19-1"></span>68. Ni, J.; Wangensteen, K.J.; Nelsen, D.; Balciunas, D.; Skuster, K.J.; Urban, M.D.; Ekker, S.C. Active recombinant Tol2 transposase for gene transfer and gene discovery applications. *Mob. DNA* **2016**, *7*, 6. [\[CrossRef\]](https://doi.org/10.1186/s13100-016-0062-z) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/27042235)
- <span id="page-19-2"></span>69. Urasaki, A.; Morvan, G.; Kawakami, K. Functional dissection of the Tol2 transposable element identified the minimal cis-sequence and a highly repetitive sequence in the subterminal region essential for transposition. *Genetics* **2006**, *174*, 639–649. [\[CrossRef\]](https://doi.org/10.1534/genetics.106.060244)
- <span id="page-19-3"></span>70. Suster, M.L.; Sumiyama, K.; Kawakami, K. Transposon-mediated BAC transgenesis in zebrafish and mice. *BMC Genomics* **2009**, *10*, 477. [\[CrossRef\]](https://doi.org/10.1186/1471-2164-10-477)
- <span id="page-19-4"></span>71. Kondrychyn, I.; Garcia-Lecea, M.; Emelyanov, A.; Parinov, S.; Korzh, V. Genome-wide analysis of Tol2 transposon reintegration in zebrafish. *BMC Genomics* **2009**, *10*, 418. [\[CrossRef\]](https://doi.org/10.1186/1471-2164-10-418) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/19737393)
- <span id="page-19-5"></span>72. Huang, P.; Xu, L.; Liang, W.; Tam, C.I.; Zhang, Y.; Qi, F.; Zhu, Z.; Lin, S.; Zhang, B. Genomic deletion induced by Tol2 transposon excision in zebrafish. *Nucleic Acids Res.* **2013**, *41*, e36. [\[CrossRef\]](https://doi.org/10.1093/nar/gks1035)
- <span id="page-19-6"></span>73. Woodard, L.E.; Li, X.; Malani, N.; Kaja, A.; Hice, R.H.; Atkinson, P.W.; Bushman, F.D.; Craig, N.L.; Wilson, M.H. Comparative Analysis of the Recently Discovered hAT Transposon TcBuster in Human Cells. *PLoS ONE* **2012**, *7*, e42666. [\[CrossRef\]](https://doi.org/10.1371/journal.pone.0042666)
- <span id="page-19-7"></span>74. Lee, J.; Jung, M.; Jeong, E.; Lee, J.K. Using Sniper-Cas9 to Minimize Off-target Effects of CRISPR-Cas9 Without the Loss of On-target Activity Via Directed Evolution. *J. Vis. Exp.* **2019**, *144*, e59202. [\[CrossRef\]](https://doi.org/10.3791/59202-v)
- <span id="page-19-8"></span>75. Kim, Y.; Kim, N.; Okafor, I.; Choi, S.; Min, S.; Lee, J.; Bae, S.-M.; Choi, K.; Choi, J.; Harihar, V.; et al. Sniper2L is a high-fidelity Cas9 variant with high activity. *Nat. Chem. Biol.* **2023**, *19*, 972–980. [\[CrossRef\]](https://doi.org/10.1038/s41589-023-01279-5) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/36894722)
- <span id="page-19-9"></span>76. Walton, R.T.; Christie, K.A.; Whittaker, M.N.; Kleinstiver, B.P. Unconstrained genome targeting with near-PAMless engineered CRISPR-Cas9 variants. *Science* **2020**, *368*, 290–296. [\[CrossRef\]](https://doi.org/10.1126/science.aba8853) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/32217751)
- <span id="page-19-10"></span>77. Roth, T.L.; Puig-Saus, C.; Yu, R.; Shifrut, E.; Carnevale, J.; Li, P.J.; Hiatt, J.; Saco, J.; Krystofinski, P.; Li, H.; et al. Reprogramming human T cell function and specificity with non-viral genome targeting. *Nature* **2018**, *559*, 405–409. [\[CrossRef\]](https://doi.org/10.1038/s41586-018-0326-5)
- <span id="page-19-11"></span>78. Zetsche, B.; Gootenberg, J.S.; Abudayyeh, O.O.; Slaymaker, I.M.; Makarova, K.S.; Essletzbichler, P.; Volz, S.E.; Joung, J.; van der Oost, J.; Regev, A.; et al. Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System. *Cell* **2015**, *163*, 759–771. [\[CrossRef\]](https://doi.org/10.1016/j.cell.2015.09.038)
- <span id="page-19-12"></span>79. McGaw, C.; Garrity, A.J.; Munoz, G.Z.; Haswell, J.R.; Sengupta, S.; Keston-Smith, E.; Hunnewell, P.; Ornstein, A.; Bose, M.; Wessells, Q.; et al. Engineered Cas12i2 is a versatile high-efficiency platform for therapeutic genome editing. *Nat. Commun.* **2022**, *13*, 2833. [\[CrossRef\]](https://doi.org/10.1038/s41467-022-30465-7) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/35595757)
- <span id="page-19-13"></span>80. Glaser, V.; Flugel, C.; Kath, J.; Du, W.; Drosdek, V.; Franke, C.; Stein, M.; Pruß, A.; Schmueck-Henneresse, M.; Volk, H.-D.; et al. Combining different CRISPR nucleases for simultaneous knock-in and base editing prevents translocations in multiplex-edited CAR T cells. *Genome Biol.* **2023**, *24*, 89. [\[CrossRef\]](https://doi.org/10.1186/s13059-023-02928-7)
- <span id="page-19-14"></span>81. Kim, H.J.; Lee, H.J.; Kim, H.; Cho, S.W.; Kim, J.-S. Targeted genome editing in human cells with zinc finger nucleases constructed via modular assembly. *Genome Res.* **2009**, *19*, 1279–1288. [\[CrossRef\]](https://doi.org/10.1101/gr.089417.108)
- <span id="page-19-15"></span>82. Moscou, M.J.; Bogdanove, A.J. A Simple Cipher Governs DNA Recognition by TAL Effectors. *Science* **2009**, *326*, 1501. [\[CrossRef\]](https://doi.org/10.1126/science.1178817) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/19933106)
- <span id="page-19-16"></span>83. Kim, Y.; Kweon, J.; Kim, A.; Chon, J.K.; Yoo, J.Y.; Kim, H.J.; Kim, S.; Lee, C.; Jeong, E.; Chung, E.; et al. A library of TAL effector nucleases spanning the human genome. *Nat. Biotechnol.* **2013**, *31*, 251–258. [\[CrossRef\]](https://doi.org/10.1038/nbt.2517) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/23417094)
- <span id="page-19-17"></span>84. Jo, S.; Das, S.; Williams, A.; Chretien, A.-S.; Pagliardini, T.; Le Roy, A.; Fernandez, J.P.; Le Clerre, D.; Jahangiri, B.; Chion-Sotinel, I.; et al. Endowing universal CAR T-cell with immune-evasive properties using TALEN-gene editing. *Nat. Commun.* **2022**, *13*, 3453. [\[CrossRef\]](https://doi.org/10.1038/s41467-022-30896-2) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/35773273)
- <span id="page-19-18"></span>85. Dharani, S.; Cho, H.; Fernandez, J.P.; Juillerat, A.; Valton, J.; Duchateau, P.; Poirot, L.; Das, S. TALEN-edited allogeneic inducible dual CAR T cells enable effective targeting of solid tumors while mitigating off-tumor toxicity. *Mol. Ther.* **2024**, *32*, 3915–3931. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2024.08.018)
- <span id="page-19-19"></span>86. Scully, R.; Panday, A.; Elango, R.; Willis, N.A. DNA double-strand break repair-pathway choice in somatic mammalian cells. *Nat. Rev. Mol. Cell Biol.* **2019**, *20*, 698–714. [\[CrossRef\]](https://doi.org/10.1038/s41580-019-0152-0) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/31263220)
- <span id="page-19-20"></span>87. Chang, H.H.Y.; Watanabe, G.; Gerodimos, C.A.; Ochi, T.; Blundell, T.L.; Jackson, S.P.; Lieber, M.R. Different DNA End Configurations Dictate Which NHEJ Components Are Most Important for Joining Efficiency. *J. Biol. Chem.* **2016**, *291*, 24377–24389. [\[CrossRef\]](https://doi.org/10.1074/jbc.M116.752329)
- <span id="page-19-21"></span>88. San Filippo, J.; Sung, P.; Klein, H. Mechanism of Eukaryotic Homologous Recombination. *Annu. Rev. Biochem.* **2008**, *77*, 229–257. [\[CrossRef\]](https://doi.org/10.1146/annurev.biochem.77.061306.125255) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/18275380)
- <span id="page-19-22"></span>89. Yeh, C.D.; Richardson, C.D.; Corn, J.E. Advances in genome editing through control of DNA repair pathways. *Nat. Cell Biol.* **2019**, *21*, 1468–1478. [\[CrossRef\]](https://doi.org/10.1038/s41556-019-0425-z) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/31792376)
- <span id="page-19-23"></span>90. Yoshimi, K.; Kunihiro, Y.; Kaneko, T.; Nagahora, H.; Voigt, B.; Mashimo, T. ssODN-mediated knock-in with CRISPR-Cas for large genomic regions in zygotes. *Nat. Commun.* **2016**, *7*, 10431. [\[CrossRef\]](https://doi.org/10.1038/ncomms10431)
- <span id="page-19-24"></span>91. Wang, C.; Fang, S.; Chen, Y.; Tang, N.; Jiao, G.; Hu, Y.; Li, J.; Shan, Q.; Wang, X.; Feng, G.; et al. High-efficiency targeted transgene integration via primed micro-homologues. *Cell Discov.* **2023**, *9*, 69. [\[CrossRef\]](https://doi.org/10.1038/s41421-023-00552-0) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37402729)
- <span id="page-19-25"></span>92. Li, Z.; You, L.; Hermann, A.; Bier, E. Developmental progression of DNA double-strand break repair deciphered by a single-allele resolution mutation classifier. *Nat. Commun.* **2024**, *15*, 2629. [\[CrossRef\]](https://doi.org/10.1038/s41467-024-46479-2) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/38521791)
- <span id="page-20-0"></span>93. Bétermier, M.; Bertrand, P.; Lopez, B.S. Is Non-Homologous End-Joining Really an Inherently Error-Prone Process? *PLoS Genet.* **2014**, *10*, e1004086. [\[CrossRef\]](https://doi.org/10.1371/journal.pgen.1004086)
- <span id="page-20-1"></span>94. Balke-Want, H.; Keerthi, V.; Gkitsas, N.; Mancini, A.G.; Kurgan, G.L.; Fowler, C.; Xu, P.; Liu, X.; Asano, K.; Patel, S.S.; et al. Homology-independent targeted insertion (HITI) enables guided CAR knock-in and efficient clinical scale CAR-T cell manufacturing. *Mol. Cancer* **2023**, *22*, 100. [\[CrossRef\]](https://doi.org/10.1186/s12943-023-01799-7) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37365642)
- <span id="page-20-2"></span>95. Cornu, T.I.; Mussolino, C.; Cathomen, T. Refining strategies to translate genome editing to the clinic. *Nat. Med.* **2017**, *23*, 415–423. [\[CrossRef\]](https://doi.org/10.1038/nm.4313) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/28388605)
- <span id="page-20-3"></span>96. Cullot, G.; Boutin, J.; Fayet, S.; Prat, F.; Rosier, J.; Cappellen, D.; Lamrissi, I.; Pennamen, P.; Bouron, J.; Amintas, S.; et al. Cell cycle arrest and p53 prevent ON-target megabase-scale rearrangements induced by CRISPR-Cas9. *Nat. Commun.* **2023**, *14*, 4072. [\[CrossRef\]](https://doi.org/10.1038/s41467-023-39632-w)
- <span id="page-20-4"></span>97. Carusillo, A.; Haider, S.; Schäfer, R.; Rhiel, M.; Türk, D.; Chmielewski, K.O.; Klermund, J.; Mosti, L.; Andrieux, G.; Schäfer, R.; et al. A novel Cas9 fusion protein promotes targeted genome editing with reduced mutational burden in primary human cells. *Nucleic Acids Res.* **2023**, *51*, 4660–4673. [\[CrossRef\]](https://doi.org/10.1093/nar/gkad255) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37070192)
- <span id="page-20-5"></span>98. Kath, J.; Du, W.; Pruene, A.; Braun, T.; Thommandru, B.; Turk, R.; Sturgeon, M.L.; Kurgan, G.L.; Amini, L.; Stein, M.; et al. Pharmacological interventions enhance virus-free generation of TRAC-replaced CAR T cells. *Mol. Ther. Methods Clin. Dev.* **2022**, *25*, 311–330. [\[CrossRef\]](https://doi.org/10.1016/j.omtm.2022.03.018)
- 99. Liang, S.; Thomas, S.E.; Chaplin, A.K.; Hardwick, S.W.; Chirgadze, D.Y.; Blundell, T.L. Structural insights into inhibitor regulation of the DNA repair protein DNA-PKcs. *Nature* **2022**, *601*, 643–648. [\[CrossRef\]](https://doi.org/10.1038/s41586-021-04274-9)
- <span id="page-20-6"></span>100. Wimberger, S.; Akrap, N.; Firth, M.; Brengdahl, J.; Engberg, S.; Schwinn, M.K.; Slater, M.R.; Lundin, A.; Hsieh, P.-P.; Li, S.; et al. Simultaneous inhibition of DNA-PK and Polθ improves integration efficiency and precision of genome editing. *Nat. Commun.* **2023**, *14*, 4761. [\[CrossRef\]](https://doi.org/10.1038/s41467-023-40344-4) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37580318)
- <span id="page-20-7"></span>101. Aznauryan, E.; Yermanos, A.; Kinzina, E.; Devaux, A.; Kapetanovic, E.; Milanova, D.; Church, G.M.; Reddy, S.T. Discovery and validation of human genomic safe harbor sites for gene and cell therapies. *Cell Reports Methods* **2022**, *2*, 100154. [\[CrossRef\]](https://doi.org/10.1016/j.crmeth.2021.100154) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/35474867)
- <span id="page-20-8"></span>102. Torikai, H.; Reik, A.; Liu, P.-Q.; Zhou, Y.; Zhang, L.; Maiti, S.; Huls, H.; Miller, J.C.; Kebriaei, P.; Rabinovitch, B.; et al. A foundation for universal T-cell based immunotherapy: T cells engineered to express a CD19-specific chimeric-antigen-receptor and eliminate expression of endogenous TCR. *Blood* **2012**, *119*, 5697–5705. [\[CrossRef\]](https://doi.org/10.1182/blood-2012-01-405365) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/22535661)
- <span id="page-20-9"></span>103. Rothemejer, F.H.; Lauritsen, N.P.; Juhl, A.K.; Schleimann, M.H.; König, S.; Søgaard, O.S.; Bak, R.O.; Tolstrup, M. Development of HIV-Resistant CAR T Cells by CRISPR/Cas-Mediated CAR Integration into the CCR5 Locus. *Viruses* **2023**, *15*, 202. [\[CrossRef\]](https://doi.org/10.3390/v15010202)
- <span id="page-20-10"></span>104. Guo, Y.; Tong, C.; Su, L.; Zhang, W.; Jia, H.; Liu, Y.; Yang, Q.; Wu, Z.; Wang, Y.; Han, W. CRISPR/Cas9 genome-edited universal CAR T cells in patients with relapsed/refractory lymphoma. *Blood Adv.* **2022**, *6*, 2695–2699. [\[CrossRef\]](https://doi.org/10.1182/bloodadvances.2021006232)
- <span id="page-20-11"></span>105. Eyquem, J.; Mansilla-Soto, J.; Giavridis, T.; van der Stegen, S.J.C.; Hamieh, M.; Cunanan, K.M.; Odak, A.; Gönen, M.; Sadelain, M. Targeting a CAR to the TRAC locus with CRISPR/Cas9 enhances tumour rejection. *Nature* **2017**, *543*, 113–117. [\[CrossRef\]](https://doi.org/10.1038/nature21405)
- <span id="page-20-12"></span>106. Kath, J.; Franke, C.; Drosdek, V.; Du, W.; Glaser, V.; Fuster-Garcia, C.; Stein, M.; Zittel, T.; Schulenberg, S.; Porter, C.E.; et al. Integration of ζ -deficient CARs into the CD3 ζ gene conveys potent cytotoxicity in T and NK cells. *Blood* **2024**, *143*, 2599–2611. [\[CrossRef\]](https://doi.org/10.1182/blood.2023020973)
- <span id="page-20-13"></span>107. Mansilla-Soto, J.; Eyquem, J.; Haubner, S.; Hamieh, M.; Feucht, J.; Paillon, N.; Zucchetti, A.E.; Li, Z.; Sjöstrand, M.; Lindenbergh, P.L.; et al. HLA-independent T cell receptors for targeting tumors with low antigen density. *Nat. Med.* **2022**, *28*, 345–352. [\[CrossRef\]](https://doi.org/10.1038/s41591-021-01621-1) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/35027758)
- <span id="page-20-14"></span>108. Höijer, I.; Emmanouilidou, A.; Östlund, R.; van Schendel, R.; Bozorgpana, S.; Tijsterman, M.; Feuk, L.; Gyllensten, U.; den Hoed, M.; Ameur, A. CRISPR-Cas9 induces large structural variants at on-target and off-target sites in vivo that segregate across generations. *Nat. Commun.* **2022**, *13*, 627. [\[CrossRef\]](https://doi.org/10.1038/s41467-022-28244-5)
- <span id="page-20-15"></span>109. Tsai, S.Q.; Zheng, Z.; Nguyen, N.T.; Liebers, M.; Topkar, V.V.; Thapar, V.; Wyvekens, N.; Khayter, C.; Iafrate, A.J.; Le, L.P.; et al. GUIDE-seq enables genome-wide profiling of off-target cleavage by CRISPR-Cas nucleases. *Nat. Biotechnol.* **2015**, *33*, 187–197. [\[CrossRef\]](https://doi.org/10.1038/nbt.3117)
- <span id="page-20-16"></span>110. Tsai, S.Q.; Nguyen, N.T.; Malagon-Lopez, J.; Topkar, V.V.; Aryee, M.J.; Joung, J.K. CIRCLE-seq: A highly sensitive in vitro screen for genome-wide CRISPR–Cas9 nuclease off-targets. *Nat. Methods* **2017**, *14*, 607–614. [\[CrossRef\]](https://doi.org/10.1038/nmeth.4278) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/28459458)
- <span id="page-20-17"></span>111. Turchiano, G.; Andrieux, G.; Klermund, J.; Blattner, G.; Pennucci, V.; el Gaz, M.; Monaco, G.; Poddar, S.; Mussolino, C.; Cornu, T.I.; et al. Quantitative evaluation of chromosomal rearrangements in gene-edited human stem cells by CAST-Seq. *Cell Stem Cell* **2021**, *28*, 1136–1147.e5. [\[CrossRef\]](https://doi.org/10.1016/j.stem.2021.02.002)
- <span id="page-20-18"></span>112. Rhiel, M.; Geiger, K.; Andrieux, G.; Rositzka, J.; Boerries, M.; Cathomen, T.; Cornu, T.I. T-CAST: An optimized CAST-Seq pipeline for TALEN confirms superior safety and efficacy of obligate-heterodimeric scaffolds. *Front. Genome Ed.* **2023**, *5*, 1130736. [\[CrossRef\]](https://doi.org/10.3389/fgeed.2023.1130736) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/36890979)
- <span id="page-20-19"></span>113. White, N.; Chalk, J.; Hu, Y.-T.; Pins, S.; Antoniou, P.; Wimberger, S.; Svensson, S.; Caetano-Silva, S.; Mudde, A.; Raj, R.; et al. MEGA dPCR reveals chromosomal aberrations, NHEJ precise repair, recurrent nuclease cleavage and DSB half-life. 2024; *preprint*. [\[CrossRef\]](https://doi.org/10.21203/rs.3.rs-4577114/v1)
- <span id="page-20-20"></span>114. Cathomen, T.; Schüle, S.; Schüßler-Lenz, M.; Abou-El-Enein, M. The Human Genome Editing Race: Loosening Regulatory Standards for Commercial Advantage? *Trends Biotechnol.* **2019**, *37*, 120–123. [\[CrossRef\]](https://doi.org/10.1016/j.tibtech.2018.06.005) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/30017092)
- <span id="page-20-21"></span>115. Ledford, H. Is CRISPR safe? Genome editing gets its first FDA scrutiny. *Nature* **2023**, *623*, 234–235. [\[CrossRef\]](https://doi.org/10.1038/d41586-023-03317-7) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37903930)
- <span id="page-21-0"></span>116. de Jong, J.; Akhtar, W.; Badhai, J.; Rust, A.G.; Rad, R.; Hilkens, J.; Berns, A.; van Lohuizen, M.; Wessels, L.F.A.; de Ridder, J. Chromatin Landscapes of Retroviral and Transposon Integration Profiles. *PLoS Genet.* **2014**, *10*, e1004250. [\[CrossRef\]](https://doi.org/10.1371/journal.pgen.1004250) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/24721906)
- <span id="page-21-1"></span>117. Moretti, A.; Ponzo, M.; Nicolette, C.A.; Tcherepanova, I.Y.; Biondi, A.; Magnani, C.F. The Past, Present, and Future of Non-Viral CAR T Cells. *Front. Immunol.* **2022**, *13*, 867013. [\[CrossRef\]](https://doi.org/10.3389/fimmu.2022.867013) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/35757746)
- <span id="page-21-2"></span>118. Bozza, M.; De Roia, A.; Correia, M.P.; Berger, A.; Tuch, A.; Schmidt, A.; Zörnig, I.; Jäger, D.; Schmidt, P.; Harbottle, R.P. A nonviral, nonintegrating DNA nanovector platform for the safe, rapid, and persistent manufacture of recombinant T cells. *Sci. Adv.* **2021**, *7*, eabf1333. [\[CrossRef\]](https://doi.org/10.1126/sciadv.abf1333)
- <span id="page-21-3"></span>119. Luke, J.M.; Vincent, J.M.; Du, S.X.; Gerdemann, U.; Leen, A.M.; Whalen, R.G.; Hodgson, C.P.; Williams, J.A. Improved antibioticfree plasmid vector design by incorporation of transient expression enhancers. *Gene Ther.* **2011**, *18*, 334–343. [\[CrossRef\]](https://doi.org/10.1038/gt.2010.149) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/21107439)
- <span id="page-21-4"></span>120. Luke, J.M.; Carnes, A.E.; Williams, J.A. Development of Antibiotic-Free Selection System for Safer DNA Vaccination. In *DNA Vaccines: Methods and Protocols*; Springer: Berlin/Heidelberg, Germany, 2014; pp. 91–111.
- <span id="page-21-5"></span>121. Williams, J.A.; Paez, P.A. Improving cell and gene therapy safety and performance using next-generation Nanoplasmid vectors. *Mol. Ther. Nucleic Acids* **2023**, *32*, 494–503. [\[CrossRef\]](https://doi.org/10.1016/j.omtn.2023.04.003)
- <span id="page-21-6"></span>122. Apolonia, L.; Waddington, S.N.; Fernandes, C.; Ward, N.J.; Bouma, G.; Blundell, M.P.; Thrasher, A.J.; Collins, M.K.; Philpott, N.J. Stable Gene Transfer to Muscle Using Non-integrating Lentiviral Vectors. *Mol. Ther.* **2007**, *15*, 1947–1954. [\[CrossRef\]](https://doi.org/10.1038/sj.mt.6300281)
- <span id="page-21-7"></span>123. Mirkovitch, J.; Mirault, M.-E.; Laemmli, U.K. Organization of the higher-order chromatin loop: Specific DNA attachment sites on nuclear scaffold. *Cell* **1984**, *39*, 223–232. [\[CrossRef\]](https://doi.org/10.1016/0092-8674(84)90208-3) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/6091913)
- <span id="page-21-8"></span>124. Wong, S.-P.; Harbottle, R.P. Genetic modification of dividing cells using episomally maintained S/MAR DNA vectors. *Mol. Ther. Nucleic Acids* **2013**, *2*, e115. [\[CrossRef\]](https://doi.org/10.1038/mtna.2013.40) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/23941867)
- <span id="page-21-9"></span>125. Jin, C.; Fotaki, G.; Ramachandran, M.; Nilsson, B.; Essand, M.; Yu, D. Safe engineering of CAR T cells for adoptive cell therapy of cancer using long-term episomal gene transfer. *EMBO Mol. Med.* **2016**, *8*, 702–711. [\[CrossRef\]](https://doi.org/10.15252/emmm.201505869) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/27189167)
- <span id="page-21-10"></span>126. Bozza, M.; Green, E.W.; Espinet, E.; De Roia, A.; Klein, C.; Vogel, V.; Offringa, R.; Williams, J.A.; Sprick, M.; Harbottle, R.P. Novel Non-integrating DNA Nano-S/MAR Vectors Restore Gene Function in Isogenic Patient-Derived Pancreatic Tumor Models. *Mol. Ther. Methods Clin. Dev.* **2020**, *17*, 957–968. [\[CrossRef\]](https://doi.org/10.1016/j.omtm.2020.04.017)
- <span id="page-21-11"></span>127. Riet, T.; Holzinger, A.; Dörrie, J.; Schaft, N.; Schuler, G.; Abken, H. Nonviral RNA Transfection to Transiently Modify T Cells with Chimeric Antigen Receptors for Adoptive Therapy. In *Synthetic Messenger RNA and Cell Metabolism Modulation: Methods and Protocols*; Springer: Berlin/Heidelberg, Germany, 2013; pp. 187–201.
- <span id="page-21-12"></span>128. Wang, Y.-S.; Kumari, M.; Chen, G.-H.; Hong, M.-H.; Yuan, J.P.-Y.; Tsai, J.-L.; Wu, H.-C. mRNA-based vaccines and therapeutics: An in-depth survey of current and upcoming clinical applications. *J. Biomed. Sci.* **2023**, *30*, 84. [\[CrossRef\]](https://doi.org/10.1186/s12929-023-00977-5)
- <span id="page-21-13"></span>129. Pascolo, S. The messenger's great message for vaccination. *Expert Rev. Vaccines* **2015**, *14*, 153–156. [\[CrossRef\]](https://doi.org/10.1586/14760584.2015.1000871)
- <span id="page-21-14"></span>130. Pardi, N.; Hogan, M.J.; Porter, F.W.; Weissman, D. mRNA vaccines—A new era in vaccinology. *Nat. Rev. Drug Discov.* **2018**, *17*, 261–279. [\[CrossRef\]](https://doi.org/10.1038/nrd.2017.243)
- <span id="page-21-15"></span>131. Foster, J.B.; Choudhari, N.; Perazzelli, J.; Storm, J.; Hofmann, T.J.; Jain, P.; Storm, P.B.; Pardi, N.; Weissman, D.; Waanders, A.J.; et al. Purification of mRNA Encoding Chimeric Antigen Receptor Is Critical for Generation of a Robust T-Cell Response. *Hum. Gene Ther.* **2019**, *30*, 168–178. [\[CrossRef\]](https://doi.org/10.1089/hum.2018.145)
- <span id="page-21-16"></span>132. Orlandini von Niessen, A.G.; Poleganov, M.A.; Rechner, C.; Plaschke, A.; Kranz, L.M.; Fesser, S.; Diken, M.; Löwer, M.; Vallazza, B.; Beissert, T.; et al. Improving mRNA-Based Therapeutic Gene Delivery by Expression-Augmenting 3′ UTRs Identified by Cellular Library Screening. *Mol. Ther.* **2019**, *27*, 824–836. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2018.12.011)
- <span id="page-21-17"></span>133. Niederer, R.O.; Rojas-Duran, M.F.; Zinshteyn, B.; Gilbert, W.V. Direct analysis of ribosome targeting illuminates thousand-fold regulation of translation initiation. *Cell Syst.* **2022**, *13*, 256–264.e3. [\[CrossRef\]](https://doi.org/10.1016/j.cels.2021.12.002) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/35041803)
- <span id="page-21-18"></span>134. Pasquinelli, A.E.; Dahlberg, J.E.; Lund, E. Reverse 5' caps in RNAs made in vitro by phage RNA polymerases. *RNA* **1995**, *1*, 957–967.
- 135. Stepinski, J.; Waddell, C.; Stolarski, R.; Darzynkiewicz, E.; Rhoads, R.E. Synthesis and properties of mRNAs containing the novel "anti-reverse" cap analogs 7-methyl(3'-O-methyl)GpppG and 7-methyl (3'-deoxy)GpppG. *RNA* **2001**, *7*, 1486–1495. [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/11680853)
- <span id="page-21-19"></span>136. Fuchs, A.-L.; Neu, A.; Sprangers, R. A general method for rapid and cost-efficient large-scale production of 5′ capped RNA. *RNA* **2016**, *22*, 1454–1466. [\[CrossRef\]](https://doi.org/10.1261/rna.056614.116)
- <span id="page-21-20"></span>137. Elango, N.; Elango, S.; Shivshankar, P.; Katz, M.S. Optimized transfection of mRNA transcribed from a d(A/T)100 tail-containing vector. *Biochem. Biophys. Res. Commun.* **2005**, *330*, 958–966. [\[CrossRef\]](https://doi.org/10.1016/j.bbrc.2005.03.067)
- 138. Holtkamp, S.; Kreiter, S.; Selmi, A.; Simon, P.; Koslowski, M.; Huber, C.; Tureci, O.; Sahin, U. Modification of antigen-encoding RNA increases stability, translational efficacy, and T-cell stimulatory capacity of dendritic cells. *Blood* **2006**, *108*, 4009–4017. [\[CrossRef\]](https://doi.org/10.1182/blood-2006-04-015024)
- <span id="page-21-21"></span>139. Passmore, L.A.; Coller, J. Roles of mRNA poly(A) tails in regulation of eukaryotic gene expression. *Nat. Rev. Mol. Cell Biol.* **2022**, *23*, 93–106. [\[CrossRef\]](https://doi.org/10.1038/s41580-021-00417-y)
- <span id="page-21-22"></span>140. Gustafsson, C.; Govindarajan, S.; Minshull, J. Codon bias and heterologous protein expression. *Trends Biotechnol.* **2004**, *22*, 346–353. [\[CrossRef\]](https://doi.org/10.1016/j.tibtech.2004.04.006) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/15245907)
- <span id="page-21-23"></span>141. Zhang, H.; Zhang, L.; Lin, A.; Xu, C.; Li, Z.; Liu, K.; Liu, B.; Ma, X.; Zhao, F.; Jiang, H.; et al. Algorithm for optimized mRNA design improves stability and immunogenicity. *Nature* **2023**, *621*, 396–403. [\[CrossRef\]](https://doi.org/10.1038/s41586-023-06127-z) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37130545)
- <span id="page-21-24"></span>142. Karikó, K.; Buckstein, M.; Ni, H.; Weissman, D. Suppression of RNA Recognition by Toll-like Receptors: The Impact of Nucleoside Modification and the Evolutionary Origin of RNA. *Immunity* **2005**, *23*, 165–175. [\[CrossRef\]](https://doi.org/10.1016/j.immuni.2005.06.008) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/16111635)
- <span id="page-22-0"></span>143. Karikó, K.; Muramatsu, H.; Welsh, F.A.; Ludwig, J.; Kato, H.; Akira, S.; Weissman, D. Incorporation of Pseudouridine Into mRNA Yields Superior Nonimmunogenic Vector With Increased Translational Capacity and Biological Stability. *Mol. Ther.* **2008**, *16*, 1833–1840. [\[CrossRef\]](https://doi.org/10.1038/mt.2008.200)
- <span id="page-22-1"></span>144. Beatty, G.L.; Haas, A.R.; Maus, M.V.; Torigian, D.A.; Soulen, M.C.; Plesa, G.; Chew, A.; Zhao, Y.; Levine, B.L.; Albelda, S.M.; et al. Mesothelin-Specific Chimeric Antigen Receptor mRNA-Engineered T Cells Induce Antitumor Activity in Solid Malignancies. *Cancer Immunol. Res.* **2014**, *2*, 112–120. [\[CrossRef\]](https://doi.org/10.1158/2326-6066.CIR-13-0170) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/24579088)
- <span id="page-22-2"></span>145. Shah, P.D.; Huang, A.C.; Xu, X.; Orlowski, R.; Amaravadi, R.K.; Schuchter, L.M.; Zhang, P.; Tchou, J.; Matlawski, T.; Cervini, A.; et al. Phase I Trial of Autologous RNA-electroporated cMET-directed CAR T Cells Administered Intravenously in Patients with Melanoma and Breast Carcinoma. *Cancer Res. Commun.* **2023**, *3*, 821–829. [\[CrossRef\]](https://doi.org/10.1158/2767-9764.CRC-22-0486)
- <span id="page-22-3"></span>146. Cummins, K.D.; Frey, N.; Nelson, A.M.; Schmidt, A.; Luger, S.; Isaacs, R.E.; Lacey, S.F.; Hexner, E.; Melenhorst, J.J.; June, C.H.; et al. Treating Relapsed/Refractory (RR) AML with Biodegradable Anti-CD123 CAR Modified T Cells. *Blood* **2017**, *130*, 1359. [\[CrossRef\]](https://doi.org/10.1182/blood.V130.Suppl_1.1359.1359)
- <span id="page-22-4"></span>147. Svoboda, J.; Rheingold, S.R.; Gill, S.I.; Grupp, S.A.; Lacey, S.F.; Kulikovskaya, I.; Suhoski, M.M.; Melenhorst, J.J.; Loudon, B.; Mato, A.R.; et al. Nonviral RNA chimeric antigen receptor–modified T cells in patients with Hodgkin lymphoma. *Blood* **2018**, *132*, 1022–1026. [\[CrossRef\]](https://doi.org/10.1182/blood-2018-03-837609)
- <span id="page-22-5"></span>148. Maus, M.V.; Haas, A.R.; Beatty, G.L.; Albelda, S.M.; Levine, B.L.; Liu, X.; Zhao, Y.; Kalos, M.; June, C.H. T Cells Expressing Chimeric Antigen Receptors Can Cause Anaphylaxis in Humans. *Cancer Immunol. Res.* **2013**, *1*, 26–31. [\[CrossRef\]](https://doi.org/10.1158/2326-6066.CIR-13-0006) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/24777247)
- <span id="page-22-6"></span>149. Foster, J.B.; Barrett, D.M.; Karikó, K. The Emerging Role of In Vitro-Transcribed mRNA in Adoptive T Cell Immunotherapy. *Mol. Ther.* **2019**, *27*, 747–756. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2019.01.018)
- <span id="page-22-7"></span>150. Paunovska, K.; Loughrey, D.; Dahlman, J.E. Drug delivery systems for RNA therapeutics. *Nat. Rev. Genet.* **2022**, *23*, 265–280. [\[CrossRef\]](https://doi.org/10.1038/s41576-021-00439-4)
- <span id="page-22-8"></span>151. Bhatia, S.N.; Dahlman, J.E. RNA delivery systems. *Proc. Natl. Acad. Sci. USA* **2024**, *121*, e2315789121. [\[CrossRef\]](https://doi.org/10.1073/pnas.2315789121)
- <span id="page-22-9"></span>152. Mackensen, A.; Haanen, J.B.A.G.; Koenecke, C.; Alsdorf, W.; Wagner-Drouet, E.; Borchmann, P.; Heudobler, D.; Ferstl, B.; Klobuch, S.; Bokemeyer, C.; et al. CLDN6-specific CAR-T cells plus amplifying RNA vaccine in relapsed or refractory solid tumors: The phase 1 BNT211-01 trial. *Nat. Med.* **2023**, *29*, 2844–2853. [\[CrossRef\]](https://doi.org/10.1038/s41591-023-02612-0) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37872225)
- <span id="page-22-10"></span>153. Chen, R.; Wang, S.K.; Belk, J.A.; Amaya, L.; Li, Z.; Cardenas, A.; Abe, B.T.; Chen, C.-K.; Wender, P.A.; Chang, H.Y. Engineering circular RNA for enhanced protein production. *Nat. Biotechnol.* **2023**, *41*, 262–272. [\[CrossRef\]](https://doi.org/10.1038/s41587-022-01393-0)
- <span id="page-22-11"></span>154. Rohner, E.; Yang, R.; Foo, K.S.; Goedel, A.; Chien, K.R. Unlocking the promise of mRNA therapeutics. *Nat. Biotechnol.* **2022**, *40*, 1586–1600. [\[CrossRef\]](https://doi.org/10.1038/s41587-022-01491-z) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/36329321)
- <span id="page-22-12"></span>155. Huang, X.; Kong, N.; Zhang, X.; Cao, Y.; Langer, R.; Tao, W. The landscape of mRNA nanomedicine. *Nat. Med.* **2022**, *28*, 2273–2287. [\[CrossRef\]](https://doi.org/10.1038/s41591-022-02061-1)
- <span id="page-22-13"></span>156. Gross, G.; Waks, T.; Eshhar, Z. Expression of immunoglobulin-T-cell receptor chimeric molecules as functional receptors with antibody-type specificity. *Proc. Natl. Acad. Sci. USA* **1989**, *86*, 10024–10028. [\[CrossRef\]](https://doi.org/10.1073/pnas.86.24.10024) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/2513569)
- <span id="page-22-14"></span>157. Kuwana, Y.; Asakura, Y.; Utsunomiya, N.; Nakanishi, M.; Arata, Y.; Itoh, S.; Nagase, F.; Kurosawa, Y. Expression of chimeric receptor composed of immunoglobulin-derived V regions and T-cell receptor-derived C regions. *Biochem. Biophys. Res. Commun.* **1987**, *149*, 960–968. [\[CrossRef\]](https://doi.org/10.1016/0006-291X(87)90502-X)
- <span id="page-22-15"></span>158. Tix, T.; Alhomoud, M.; Shouval, R.; Cliff, E.R.S.; Perales, M.-A.; Cordas dos Santos, D.M.; Rejeski, K. Second primary malignancies after CAR T-cell therapy: A systematic review and meta-analysis of 5,517 lymphoma and myeloma patients. *Clin. Cancer Res.* **2024**, *30*, 4690–4700. [\[CrossRef\]](https://doi.org/10.1158/1078-0432.CCR-24-1798)
- <span id="page-22-16"></span>159. Levine, B.L.; Pasquini, M.C.; Connolly, J.E.; Porter, D.L.; Gustafson, M.P.; Boelens, J.J.; Horwitz, E.M.; Grupp, S.A.; Maus, M.V.; Locke, F.L.; et al. Unanswered questions following reports of secondary malignancies after CAR-T cell therapy. *Nat. Med.* **2024**, *30*, 338–341. [\[CrossRef\]](https://doi.org/10.1038/s41591-023-02767-w)
- <span id="page-22-17"></span>160. Hacein-Bey-Abina, S.; Garrigue, A.; Wang, G.P.; Soulier, J.; Lim, A.; Morillon, E.; Clappier, E.; Caccavelli, L.; Delabesse, E.; Beldjord, K.; et al. Insertional oncogenesis in 4 patients after retrovirus-mediated gene therapy of SCID-X1. *J. Clin. Invest.* **2008**, *118*, 3132–3142. [\[CrossRef\]](https://doi.org/10.1172/JCI35700)
- <span id="page-22-18"></span>161. Hausl, M.; Zhang, W.; Voigtlander, R.; Muther, N.; Rauschhuber, C.; Ehrhardt, A. Development of Adenovirus Hybrid Vectors for Sleeping Beauty Transposition in Large Mammals. *Curr. Gene Ther.* **2011**, *11*, 363–374. [\[CrossRef\]](https://doi.org/10.2174/156652311797415890) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/21888620)
- 162. Zhang, W.; Muck-Hausl, M.; Wang, J.; Sun, C.; Gebbing, M.; Miskey, C.; Ivics, Z.; Izsvak, Z.; Ehrhardt, A. Integration Profile and Safety of an Adenovirus Hybrid-Vector Utilizing Hyperactive Sleeping Beauty Transposase for Somatic Integration. *PLoS ONE* **2013**, *8*, e75344. [\[CrossRef\]](https://doi.org/10.1371/journal.pone.0075344)
- 163. Cunningham, S.C.; Siew, S.M.; Hallwirth, C.V.; Bolitho, C.; Sasaki, N.; Garg, G.; Michael, I.P.; Hetherington, N.A.; Carpenter, K.; de Alencastro, G.; et al. Modeling correction of severe urea cycle defects in the growing murine liver using a hybrid recombinant adeno-associated virus/piggyBac transposase gene delivery system. *Hepatology* **2015**, *62*, 417–428. [\[CrossRef\]](https://doi.org/10.1002/hep.27842)
- 164. Sandoval-Villegas, N.; Ivics, Z. The best of both worlds: AAV-mediated gene transfer empowered by LNP delivery of Sleeping Beauty transposase for durable transgene expression in vivo. *Mol. Ther.* **2024**, *32*, 3211–3214. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2024.09.002) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/39326408)
- <span id="page-22-19"></span>165. Li, C.; Psatha, N.; Wang, H.; Singh, M.; Samal, H.B.; Zhang, W.; Ehrhardt, A.; Izsvák, Z.; Papayannopoulou, T.; Lieber, A. Integrating HDAd5/35++ Vectors as a New Platform for HSC Gene Therapy of Hemoglobinopathies. *Mol. Ther. Methods Clin. Dev.* **2018**, *9*, 142–152. [\[CrossRef\]](https://doi.org/10.1016/j.omtm.2018.02.004) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/29766024)
- <span id="page-23-0"></span>166. Zhang, X.; Van Treeck, B.; Horton, C.A.; McIntyre, J.J.R.; Palm, S.M.; Shumate, J.L.; Collins, K. *Harnessing Eukaryotic Retroelement Proteins for Transgene Insertion into Human Safe-Harbor Loci*; Springer US: Berlin/Heidelberg, Germany, 2024; ISBN 4158702402137.
- <span id="page-23-1"></span>167. Chen, P.J.; Liu, D.R. Prime editing for precise and highly versatile genome manipulation. *Nat. Rev. Genet.* **2023**, *24*, 161–177. [\[CrossRef\]](https://doi.org/10.1038/s41576-022-00541-1) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/36344749)
- <span id="page-23-2"></span>168. Fiumara, M.; Ferrari, S.; Omer-Javed, A.; Beretta, S.; Albano, L.; Canarutto, D.; Varesi, A.; Gaddoni, C.; Brombin, C.; Cugnata, F.; et al. Genotoxic effects of base and prime editing in human hematopoietic stem cells. *Nat. Biotechnol.* **2024**, *42*, 877–891. [\[CrossRef\]](https://doi.org/10.1038/s41587-023-01915-4) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37679541)
- <span id="page-23-3"></span>169. Xiang, J.; Devenport, J.M.; Carter, A.J.; Staser, K.W.; Kim, M.Y.; O' Neal, J.; Ritchey, J.K.; Rettig, M.P.; Gao, F.; Rettig, G.; et al. An "off-the-shelf" CD2 universal CAR-T therapy for T-cell malignancies. *Leukemia* **2023**, *37*, 2448–2456. [\[CrossRef\]](https://doi.org/10.1038/s41375-023-02039-z) [\[PubMed\]](https://www.ncbi.nlm.nih.gov/pubmed/37798328)
- <span id="page-23-4"></span>170. Zhang, H.-X.; Zhang, Y.; Yin, H. Genome Editing with mRNA Encoding ZFN, TALEN, and Cas9. *Mol. Ther.* **2019**, *27*, 735–746. [\[CrossRef\]](https://doi.org/10.1016/j.ymthe.2019.01.014)
- <span id="page-23-5"></span>171. Ku, K.S.; Tang, J.; Chen, Y.; Shi, Y. Current Advancements in Anti-Cancer Chimeric Antigen Receptor T Cell Immunotherapy and How Nanotechnology May Change the Game. *Int. J. Mol. Sci.* **2024**, *25*, 5361. [\[CrossRef\]](https://doi.org/10.3390/ijms25105361)
- <span id="page-23-6"></span>172. Cheng, Q.; Wei, T.; Farbiak, L.; Johnson, L.T.; Dilliard, S.A.; Siegwart, D.J. Selective organ targeting (SORT) nanoparticles for tissue-specific mRNA delivery and CRISPR–Cas gene editing. *Nat. Nanotechnol.* **2020**, *15*, 313–320. [\[CrossRef\]](https://doi.org/10.1038/s41565-020-0669-6)
- <span id="page-23-7"></span>173. Kon, E.; Ad-El, N.; Hazan-Halevy, I.; Stotsky-Oterin, L.; Peer, D. Targeting cancer with mRNA–lipid nanoparticles: Key considerations and future prospects. *Nat. Rev. Clin. Oncol.* **2023**, *20*, 739–754. [\[CrossRef\]](https://doi.org/10.1038/s41571-023-00811-9)

**Disclaimer/Publisher's Note:** The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.